<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Briney Lab </title> <meta name="author" content="Briney Lab"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="antibody, immunology, computational immunology, structure, function, evolution"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A5%BC&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="http://localhost:4000/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Briney</span> Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/"> </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/code/">code </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Burbach2025-ej" class="col-sm-8"> <div class="title">A curriculum learning approach to training antibody language models</div> <div class="author"> Sarah M Burbach and <em>Bryan Briney</em> </div> <div class="periodical"> <em>PLoS computational biology</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pcbi.1013473" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>There is growing interest in pre-training antibody language models (AbLMs) with a mixture of unpaired and natively paired sequences, seeking to combine the proven benefits of training with natively paired sequences with the massive scale of unpaired antibody sequence datasets. However, given the novelty of this strategy, the field lacks a systematic evaluation of data processing methods and training strategies that maximize the benefits of mixed training data while accommodating the significant imbalance in the size of existing paired and unpaired datasets. Here, we introduce a method of curriculum learning for AbLMs, which facilitates a gradual transition from unpaired to paired sequences during training. We optimize this method and compare it to other data sampling strategies for AbLMs, including a constant mix and a fine-tuning approach. We observe that the curriculum and constant approaches show improved performance compared to the fine-tuning approach in large-scale models, likely due to their ability to prevent catastrophic forgetting and slow overfitting. Finally, we show that a 650M-parameter curriculum model, CurrAb, outperforms existing mixed AbLMs in downstream residue prediction and classification tasks.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Neyestanak2025-gr" class="col-sm-8"> <div class="title">Data-optimal scaling of paired antibody language models</div> <div class="author"> Mahdi Shafiei Neyestanak, Sarah M Burbach, Karenna Ng, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Praneeth Gangavarapu, Jonathan Hurtado, Judie Magura, Nasreen Ismail, Daniel Muema, Thumbi Ndung’u, Andrew B Ward, Bryan Briney' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2025.09.02.673765" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Scaling laws for large language models in natural language domains are typically derived under the assumption that performance is primarily compute-constrained. In contrast, antibody language models (AbLMs) trained on paired sequences are primarily data-limited, thus requiring different considerations. To explore how model size and data scale affect AbLM performance, we trained 15 AbLMs across all pairwise combinations of five model sizes and three training data sizes. From these experiments, we derive an AbLM-specific scaling law and estimate that training a data-optimal AbLM equivalent of the highly performant 650M-parameter ESM-2 protein language model would require ∼5.5 million paired antibody sequences. Evaluation on multiple downstream classification tasks revealed that significant performance gains emerged only with sufficiently large model size, suggesting that in data-limited domains, improved performance depends jointly on both model scale and data volume.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Pratap2025-yw" class="col-sm-8"> <div class="title">Immunofocusing on the conserved fusion peptide of HIV envelope glycoprotein in rhesus macaques</div> <div class="author"> Payal P Pratap, Christopher A Cottrell, James Quinn, and <span class="more-authors" title="click to view 23 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '23 more authors' ? 'Diane G Carnathan, Daniel L V Bader, Andy S Tran, Chiamaka A Enemuo, Julia T Ngo, Sara T Richey, Hongmei Gao, Xiaoying Shen, Kelli M Greene, Jonathan Hurtado, Katarzyna Kaczmarek Michaels, Elana Ben-Akiva, Ashley Lemnios, Mariane B Melo, Joel D Allen, Gabriel Ozorowski, Max Crispin, Bryan Briney, David Montefiori, Guido Silvestri, Darrell J Irvine, Shane Crotty, Andrew B Ward' : '23 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">23 more authors</span> </div> <div class="periodical"> <em>npj vaccines</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41541-025-01252-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>During infection, the fusion peptide (FP) of HIV envelope glycoprotein (Env) serves a central role in viral fusion with the host cell. As such, the FP is highly conserved and therefore an attractive epitope for vaccine design. Here, we describe a vaccination study in non-human primates (NHPs) where glycan deletions were made on soluble HIV Env to increase FP epitope exposure. When delivered via implantable osmotic pumps, this immunogen primed immune responses against the FP, which were then boosted with heterologous trimers resulting in a focused immune response targeting the conserved FP epitope. Although autologous immunizations did not elicit high affinity FP-targeting antibodies, the conserved FP epitope on a heterologous trimer further matured the lower affinity, FP-targeting B cells. This study suggests using epitope conservation strategies on distinct Env trimer immunogens can focus humoral responses on desired neutralizing epitopes and suppress immune-distracting antibody responses against non-neutralizing epitopes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Song2025-zf" class="col-sm-8"> <div class="title">Broadly neutralizing antibodies targeting a conserved silent face of spike RBD resist extreme SARS-CoV-2 antigenic drift</div> <div class="author"> Ge Song, Meng Yuan, Hejun Liu, and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Tazio Capozzola, Ryan N Lin, Jonathan L Torres, Wan-Ting He, Rami Musharrafieh, Katharina Dueker, Panpan Zhou, Sean Callaghan, Nitesh Mishra, Peter Yong, Fabio Anzanello, Gabriel Avillion, Anh Lina Vo, Xuduo Li, Yuexiu Zhang, Muzamil Makhdoomi, Ziqi Feng, Xueyong Zhu, Linghang Peng, David Nemazee, Yana Safonova, Bryan Briney, Andrew B Ward, Dennis R Burton, Ian A Wilson, Raiees Andrabi' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>Cell reports</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2025.115948" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Developing broad coronavirus vaccines hinges on identifying and understanding the molecular basis of conserved spike epitopes targeted by broadly neutralizing antibodies (bnAbs). Building on our earlier work identifying sarbecovirus receptor-binding domain (RBD) group 1 and 2 bnAbs, we now show that several of these antibodies retain neutralizing activity against highly mutated SARS-CoV-2 variants, including BA.2.86 and JN.1. Structural studies reveal that group 1 bnAbs use recurrent germline-encoded heavy-chain complementarity-determining region 3 (CDRH3) features to interact with a conserved RBD region that overlaps with class 4 bnAb site. Group 2 bnAbs recognize a less well-defined “site V” on the RBD and destabilize spike trimer. Notably, site V remains largely unchanged across SARS-CoV-2 variants and is conserved among diverse sarbecoviruses, highlighting its potential as a broad vaccine target. Our findings underscore the need for targeted vaccine strategies to induce immunofocused B cell responses to escape resistant subdominant spike RBD bnAb epitopes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Maciel2025-rh" class="col-sm-8"> <div class="title">Optimizing human B cell repertoire analyses to interpret clinical data and design sequential HIV vaccines</div> <div class="author"> Milton Maciel, Lindsey R Baden, Brian Briney, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Nicole A Doria-Rose, Kasalina N Kiwanuka, Shelly J Krebs, Angela Malaspina, Georgia D Tomaras, Andrew Ward, Gunilla B Karlsson Hedestam, Wilton B Williams, M Patricia D’Souza' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>npj vaccines</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a href="https://doi.org/10.1038/s41541-025-01168-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Ng2025-av" class="col-sm-8"> <div class="title">Focused learning by antibody language models using preferential masking of non-templated regions</div> <div class="author"> Karenna Ng and <em>Bryan Briney</em> </div> <div class="periodical"> <em>Patterns (New York, N.Y.)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.patter.2025.101239" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Existing antibody language models (AbLMs) are pre-trained using a masked language modeling (MLM) objective with uniform masking probabilities. While these models excel at predicting germline residues, they often struggle with mutated and non-templated residues, which concentrate in the complementarity-determining regions (CDRs) and are crucial for antigen binding specificity. Here, we demonstrate that preferential masking of the primarily non-templated CDR3 is a compute-efficient strategy to enhance model performance. We pre-trained two AbLMs using either uniform or preferential masking and observed that the latter improves residue prediction accuracy in the highly variable CDR3. Preferential masking also improves antibody classification by native chain pairing and binding specificity, suggesting improved CDR3 understanding and indicating that non-random, learnable patterns help govern antibody chain pairing. We further show that specificity classification is largely informed by residues in the CDRs, demonstrating that AbLMs learn meaningful patterns that align with immunological understanding.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2024-do" class="col-sm-8"> <div class="title">Conformational ensemble-based framework enables rapid development of Lassa virus vaccine candidates</div> <div class="author"> Nitesh Mishra, Gabriel Avillion, Sean Callaghan, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Charlotte DiBiase, Jonathan Hurtado, Nathan Liendo, Sarah Burbach, Terrence Messmer, Bryan Briney' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.11.21.624760" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Lassa virus (LASV), an arenavirus endemic to West Africa, poses a significant public health threat due to its high pathogenicity and expanding geographic risk zone. LASV glycoprotein complex (GPC) is the only known target of neutralizing antibodies, but its inherent metastability and conformational flexibility have hindered the development of GPC-based vaccines. We employed a variant of AlphaFold2 (AF2), called subsampled AF2, to generate diverse structures of LASV GPC that capture an array of potential conformational states using MSA subsampling and dropout layers. Conformational ensembles identified several metamorphic domains-areas of significant conformational flexibility-that could be targeted to stabilize the GPC in its immunogenic prefusion state. ProteinMPNN was then used to redesign GPC sequences to minimize the mobility of metamorphic domains. These redesigned sequences were further filtered using subsampled AF2, leading to the identification of promising GPC variants for further testing. A small library of redesigned GPC sequences was experimentally validated and showed significantly increased protein yields compared to controls. Antigenic profiles indicated these variants preserved essential epitopes for effective immune response, suggesting their potential for broad protective efficacy. Our results demonstrate that AI-driven approaches can predict the conformational landscape of complex pathogens. This knowledge can be used to stabilize viral proteins, such as LASV GPC, in their prefusion conformation, optimizing them for stability and expression, and offering a streamlined framework for vaccine design. Our deep learning / machine learning enabled framework contributes to global efforts to combat LASV and has broader implications for vaccine design and pandemic preparedness.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2024-jx" class="col-sm-8"> <div class="title">Decoding protein dynamicity in DNA ligase activity through deep learning-based structural ensembles</div> <div class="author"> Nitesh Mishra, Sean Callaghan, and <em>Bryan Briney</em> </div> <div class="periodical"> <em>bioRxiv</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2024.11.07.622521" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Numerous proteins perform their functions by transitioning between various structures. Understanding the conformational ensembles associated with these states is essential for uncovering crucial mechanistic aspects that regulate protein function. In this study, we utilized AlphaFold3 (AF3) to investigate the structural dynamics and mechanisms of enzymes involved in DNA homeostasis, using NAD-dependent Taq ligases and human DNA Ligase 1 as a case example. Modifying the input parameters for AF3 yielded detailed conformational states of a DNA-binding enzyme, thereby offering enhanced mechanistic insights. We applied AF3 to model the various stages of thermophilic Taq DNA ligase activity, from its ground state to substrate-bound complexes, revealing significant mobility in the N-terminal adenylation and C-terminal BRCT domains. These detailed structural ensembles provided novel insights into the enzyme’s behavior during DNA repair, underscoring the potential of AF3 in elucidating mechanistic details critical for therapeutic and biotechnological targeting. Extending this approach to human LIG1, we examined its end-joining activity on double-strand breaks (DSBs) with short 3’ and 5’ overhangs. In alignment with published experimental data, AF3 conformational ensembles indicated LIG1 has lower catalytic efficiency for 5’ overhangs due to suboptimal DNA positioning within the catalytic site, demonstrating AF3’s capability to capture subtle yet functionally significant conformational differences by generating conformational ensembles capturing greater structural variance.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Hurtado2024-nc" class="col-sm-8"> <div class="title">Deep repertoire mining uncovers ultra-broad coronavirus neutralizing antibodies targeting multiple spike epitopes</div> <div class="author"> Jonathan Hurtado, Thomas F Rogers, David B Jaffe, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Bruce A Adams, Sandhya Bangaru, Elijah Garcia, Tazio Capozzola, Terrence Messmer, Pragati Sharma, Ge Song, Nathan Beutler, Wanting He, Katharina Dueker, Rami Musharrafieh, Sarah Burbach, Alina Truong, Michael J T Stubbington, Dennis R Burton, Raiees Andrabi, Andrew B Ward, Wyatt J McDonnell, Bryan Briney' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Cell reports</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2024.114307" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The development of vaccines and therapeutics that are broadly effective against known and emergent coronaviruses is an urgent priority. We screened the circulating B cell repertoires of COVID-19 survivors and vaccinees to isolate over 9,000 severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-specific monoclonal antibodies (mAbs), providing an expansive view of the SARS-CoV-2-specific Ab repertoire. Among the recovered antibodies was TXG-0078, an N-terminal domain (NTD)-specific neutralizing mAb that recognizes diverse alpha- and beta-coronaviruses. TXG-0078 achieves its exceptional binding breadth while utilizing the same VH1-24 variable gene signature and heavy-chain-dominant binding pattern seen in other NTD-supersite-specific neutralizing Abs with much narrower specificity. We also report CC24.2, a pan-sarbecovirus neutralizing antibody that targets a unique receptor-binding domain (RBD) epitope and shows similar neutralization potency against all tested SARS-CoV-2 variants, including BQ.1.1 and XBB.1.5. A cocktail of TXG-0078 and CC24.2 shows protection in vivo, suggesting their potential use in variant-resistant therapeutic Ab cocktails and as templates for pan-coronavirus vaccine design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Schiffner2024-pc" class="col-sm-8"> <div class="title">Vaccination induces broadly neutralizing antibody precursors to HIV gp41</div> <div class="author"> Torben Schiffner, Ivy Phung, Rashmi Ray, and <span class="more-authors" title="click to view 72 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '72 more authors' ? 'Adriana Irimia, Ming Tian, Olivia Swanson, Jeong Hyun Lee, Chang-Chun D Lee, Ester Marina-Zárate, So Yeon Cho, Jiachen Huang, Gabriel Ozorowski, Patrick D Skog, Andreia M Serra, Kimmo Rantalainen, Joel D Allen, Sabyasachi Baboo, Oscar L Rodriguez, Sunny Himansu, Jianfu Zhou, Jonathan Hurtado, Claudia T Flynn, Katherine McKenney, Colin Havenar-Daughton, Swati Saha, Kaitlyn Shields, Steven Schultze, Melissa L Smith, Chi-Hui Liang, Laura Toy, Simone Pecetta, Ying-Cing Lin, Jordan R Willis, Fabian Sesterhenn, Daniel W Kulp, Xiaozhen Hu, Christopher A Cottrell, Xiaoya Zhou, Jennifer Ruiz, Xuesong Wang, Usha Nair, Kathrin H Kirsch, Hwei-Ling Cheng, Jillian Davis, Oleksandr Kalyuzhniy, Alessia Liguori, Jolene K Diedrich, Julia T Ngo, Vanessa Lewis, Nicole Phelps, Ryan D Tingle, Skye Spencer, Erik Georgeson, Yumiko Adachi, Michael Kubitz, Saman Eskandarzadeh, Marc A Elsliger, Rama R Amara, Elise Landais, Bryan Briney, Dennis R Burton, Diane G Carnathan, Guido Silvestri, Corey T Watson, John R Yates, James C Paulson, Max Crispin, Gevorg Grigoryan, Andrew B Ward, Devin Sok, Frederick W Alt, Ian A Wilson, Facundo D Batista, Shane Crotty, William R Schief' : '72 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">72 more authors</span> </div> <div class="periodical"> <em>Nature immunology</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>A key barrier to the development of vaccines that induce broadly neutralizing antibodies (bnAbs) against human immunodeficiency virus (HIV) and other viruses of high antigenic diversity is the design of priming immunogens that induce rare bnAb-precursor B cells. The high neutralization breadth of the HIV bnAb 10E8 makes elicitation of 10E8-class bnAbs desirable; however, the recessed epitope within gp41 makes envelope trimers poor priming immunogens and requires that 10E8-class bnAbs possess a long heavy chain complementarity determining region 3 (HCDR3) with a specific binding motif. We developed germline-targeting epitope scaffolds with affinity for 10E8-class precursors and engineered nanoparticles for multivalent display. Scaffolds exhibited epitope structural mimicry and bound bnAb-precursor human naive B cells in ex vivo screens, protein nanoparticles induced bnAb-precursor responses in …</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Cottrell2024-qy" class="col-sm-8"> <div class="title">Heterologous prime-boost vaccination drives early maturation of HIV broadly neutralizing antibody precursors in humanized mice</div> <div class="author"> Christopher A Cottrell, Xiaozhen Hu, Jeong Hyun Lee, and <span class="more-authors" title="click to view 34 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '34 more authors' ? 'Patrick Skog, Sai Luo, Claudia T Flynn, Katherine R McKenney, Jonathan Hurtado, Oleksandr Kalyuzhniy, Alessia Liguori, Jordan R Willis, Elise Landais, Sebastian Raemisch, Xuejun Chen, Sabyasachi Baboo, Sunny Himansu, Jolene K Diedrich, Hongying Duan, Cheng Cheng, Torben Schiffner, Daniel L V Bader, Daniel W Kulp, Ryan Tingle, Erik Georgeson, Saman Eskandarzadeh, Nushin Alavi, Danny Lu, Troy Sincomb, Michael Kubitz, Tina-Marie Mullen, John R Yates, James C Paulson, John R Mascola, Frederick W Alt, Bryan Briney, Devin Sok, William R Schief' : '34 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">34 more authors</span> </div> <div class="periodical"> <em>Science translational medicine</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/scitranslmed.adn0223" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A protective HIV vaccine will likely need to induce broadly neutralizing antibodies (bnAbs). Vaccination with the germline-targeting immunogen eOD-GT8 60mer adjuvanted with AS01B was found to induce VRC01-class bnAb precursors in 97% of vaccine recipients in the IAVI G001 phase 1 clinical trial; however, heterologous boost immunizations with antigens more similar to the native glycoprotein will be required to induce bnAbs. Therefore, we designed core-g28v2 60mer, a nanoparticle immunogen to be used as a first boost after eOD-GT8 60mer priming. We found, using a humanized mouse model approximating human conditions of VRC01-class precursor B cell diversity, affinity, and frequency, that both protein- and mRNA-based heterologous prime-boost regimens induced VRC01-class antibodies that gained key mutations and bound to near-native HIV envelope trimers lacking the N276 glycan. We further showed that VRC01-class antibodies induced by mRNA-based regimens could neutralize pseudoviruses lacking the N276 glycan. These results demonstrated that heterologous boosting can drive maturation toward VRC01-class bnAb development and supported the initiation of the IAVI G002 phase 1 trial testing mRNA-encoded nanoparticle prime-boost regimens.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Burbach2024-ji" class="col-sm-8"> <div class="title">Improving antibody language models with native pairing</div> <div class="author"> Sarah M Burbach and <em>Bryan Briney</em> </div> <div class="periodical"> <em>Patterns (New York, N.Y.)</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.patter.2024.100967" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Existing antibody language models are limited by their use of unpaired antibody sequence data. A recently published dataset of ∼1.6 × 106 natively paired human antibody sequences offers a unique opportunity to evaluate how antibody language models are improved by training with native pairs. We trained three baseline antibody language models (BALM), using natively paired (BALM-paired), randomly-paired (BALM-shuffled), or unpaired (BALM-unpaired) sequences from this dataset. To address the paucity of paired sequences, we additionally fine-tuned ESM (evolutionary scale modeling)-2 with natively paired antibody sequences (ft-ESM). We provide evidence that training with native pairs allows the model to learn immunologically relevant features that span the light and heavy chains, which cannot be simulated by training with random pairs. We additionally show that training with native pairs improves model performance on a variety of metrics, including the ability of the model to classify antibodies by pathogen specificity.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Kumar2024-gl" class="col-sm-8"> <div class="title">B cell repertoire sequencing of HIV-1 pediatric elite-neutralizers identifies multiple broadly neutralizing antibody clonotypes</div> <div class="author"> Sanjeev Kumar, Prashant Bajpai, Collin Joyce, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Sushil Kumar Kabra, Rakesh Lodha, Dennis R Burton, Bryan Briney, Kalpana Luthra' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Frontiers in immunology</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fimmu.2024.1272493" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Introduction: A limited subset of HIV-1 infected adult individuals typically after at least 2-3 years of chronic infection, develop broadly neutralizing antibodies (bnAbs), suggesting that highly conserved neutralizing epitopes on the HIV-1 envelope glycoprotein are difficult for B cell receptors to effectively target, during natural infection. Recent studies have shown the evolution of bnAbs in HIV-1 infected infants. Methods: We used bulk BCR sequencing (BCR-seq) to profile the B cell receptors from longitudinal samples (3 time points) collected from a rare pair of antiretroviralnaïve, HIV-1 infected pediatric monozygotic twins (AIIMS_329 and AIIMS_330) who displayed elite plasma neutralizing activity against HIV-1. Results: BCR-seq of both twins revealed convergent antibody characteristics including V-gene use, CDRH3 lengths and somatic hypermutation (SHM). Further, antibody clonotypes with genetic features similar to highly potent bnAbs isolated from adults showed ongoing development in donor AIIMS_330 but not in AIIMS_329, corroborating our earlier findings based on plasma bnAbs responses. An increase in SHM was observed in sequences of the IgA isotype from AIIMS_330. Discussion: This study suggests that children living with chronic HIV-1 can develop clonotypes of HIV-1 bnAbs against multiple envelope epitopes similar to those isolated from adults, highlighting that such B cells could be steered to elicit bnAbs responses through vaccines aimed to induce bnAbs against HIV-1 in a broad range of people including children.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Musharrafieh2023-tf" class="col-sm-8"> <div class="title">Germline-targeting chimpanzee SIV Envelopes induce V2-apex broadly neutralizing-like B cell precursors in a rhesus macaque infection model</div> <div class="author"> Rami Musharrafieh, Yana Safonova, Ge Song, and <span class="more-authors" title="click to view 42 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '42 more authors' ? 'Ryan S Roark, Fang-Hua Lee, Shiyu Zhang, Jonathan Hurtado, Peter Yong, Shuyi Wang, Ronnie M Russell, Wenge Ding, Yingying Li, Juliette Rando, Alexander I Murphy, Emily Lindemuth, Chengyan Zhao, Andrew Jesse Connell, Wen-Hsin Lee, Nitesh Mishra, Gabriel Avillion, Wanting He, Sean Callaghan, Katharina Dueker, Anh L Vo, Xuduo Li, Tazio Capozzola, Collin Joyce, Fangzhu Zhao, Fabio Anzanello, Weimin Liu, Frederic Bibollet-Ruche, Alejandra Ramos, Hui Li, Mark G Lewis, Gabriel Ozorowski, Elise Landais, Brian T Foley, Kshitij Wagh, Devin Sok, Bryan Briney, Andrew B Ward, Beatrice H Hahn, Dennis R Burton, George M Shaw, Raiees Andrabi' : '42 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">42 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2023.09.21.558743" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>SUMMARYEliciting broadly neutralizing antibodies-(bnAbs) remains a major goal of HIV-1 vaccine research. Previously, we showed that a soluble chimpanzee SIV Envelope-(Env) trimer, MT145K, bound several human V2-apex bnAb-precursors and stimulated an appropriate response in V2-apex bnAb precursor-expressing knock-in mice. Here, we tested the immunogenicity of three MT145 variants (MT145, MT145K, MT145K.dV5) expressed as chimeric simian-chimpanzee-immunodeficiency-viruses-(SCIVs) in rhesus macaques-(RMs). All three viruses established productive infections with high setpoint vRNA titers. RMs infected with the germline-targeting SCIV_MT145K and SCIV_MT145K.dV5 exhibited larger and more clonally expanded B cell lineages featuring long anionic heavy chain complementary-determining-regions-(HCDR3s) compared with wildtype SCIV_MT145. Moreover, antigen-specific B cell analysis revealed enrichment for long-CDHR3-bearing antibodies in SCIV_MT145K.dV5 infected animals with paratope features resembling prototypic V2-apex bnAbs and their precursors. Although none of the animals developed bnAbs, these results show that germline-targeting SCIVs can activate and preferentially expand B cells expressing V2-apex bnAb-like precursors, the first step in bnAb elicitation.Graphical Abstract</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2023-mp" class="col-sm-8"> <div class="title">AntiRef: reference clusters of human antibody sequences</div> <div class="author"> <em>Bryan Briney</em> </div> <div class="periodical"> <em>Bioinformatics advances</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/bioadv/vbad109" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Motivation: Genetic biases in the human antibody repertoire result in publicly available antibody sequence datasets that contain many duplicate or highly similar sequences. Available datasets are further skewed by the predominance of studies focused on specific disease states, primarily cancer, autoimmunity, and a small number of infectious diseases that includes HIV, influenza, and SARS-CoV-2. These biases and redundancies are a barrier to rapid similarity searches and reduce the efficiency with which these datasets can be used to train statistical or machine-learning models. Identity-based clustering provides a solution; however, the extremely large size of available antibody sequence datasets makes such clustering operations computationally intensive and potentially out of reach for many scientists and researchers who would benefit from such data. Results: Antibody Reference Clusters (AntiRef), which is modeled after UniRef, provides clustered datasets of filtered human antibody sequences. Due to the modular nature of recombined antibody genes, the clustering thresholds used by UniRef for general protein sequences are suboptimal for antibody clustering. Starting with an input dataset of ∼451M full-length, productive human antibody sequences, AntiRef provides reference datasets clustered at a range of antibody-optimized identity thresholds. AntiRef90 is one-third the size of the input dataset and less than half the size of the non-redundant AntiRef100. Availability and implementation: AntiRef datasets are available on Zenodo (zenodo.org/record/7474336). All code used to generate AntiRef is available on GitHub (github.com/briney/antiref). The AntiRef versioning scheme (current version: v2022.12.14) refers to the date on which sequences were retrieved from OAS.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Zhao2023-yc" class="col-sm-8"> <div class="title">Broadening a SARS-CoV-1-neutralizing antibody for potent SARS-CoV-2 neutralization through directed evolution</div> <div class="author"> Fangzhu Zhao, Meng Yuan, Celina Keating, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Namir Shaabani, Oliver Limbo, Collin Joyce, Jordan Woehl, Shawn Barman, Alison Burns, Quoc Tran, Xueyong Zhu, Michael Ricciardi, Linghang Peng, Jessica Smith, Deli Huang, Bryan Briney, Devin Sok, David Nemazee, John R Teijaro, Ian A Wilson, Dennis R Burton, Joseph G Jardine' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Science signaling</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/scisignal.abk3516" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) underscores the need for strategies to rapidly develop neutralizing monoclonal antibodies that can function as prophylactic and therapeutic agents and to help guide vaccine design. Here, we demonstrate that engineering approaches can be used to refocus an existing antibody that neutralizes one virus but not a related virus. Through a rapid affinity maturation strategy, we engineered CR3022, a SARS-CoV-1-neutralizing antibody, to bind to the receptor binding domain of SARS-CoV-2 with &gt;1000-fold increased affinity. The engineered CR3022 neutralized SARS-CoV-2 and provided prophylactic protection from viral challenge in a small animal model of SARS-CoV-2 infection. Deep sequencing throughout the engineering process paired with crystallographic analysis of engineered CR3022 elucidated the molecular mechanisms by which the antibody can accommodate sequence differences in the epitopes between SARS-CoV-1 and SARS-CoV-2. This workflow provides a blueprint for the rapid broadening of neutralization of an antibody from one virus to closely related but resistant viruses.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Joyce2023-br" class="col-sm-8"> <div class="title">Antigen pressure from two founder viruses induces multiple insertions at a single antibody position to generate broadly neutralizing HIV antibodies</div> <div class="author"> Collin Joyce, Sasha Murrell, Ben Murrell, and <span class="more-authors" title="click to view 27 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '27 more authors' ? 'Oluwarotimi Omorodion, Lorena S Ver, Nancy Carrico, Raiza Bastidas, Rebecca Nedellec, Michael Bick, Jordan Woehl, Fangzhu Zhao, Alison Burns, Shawn Barman, Michael Appel, Alejandra Ramos, Lalinda Wickramasinghe, Kemal Eren, Thomas Vollbrecht, Davey M Smith, Sergei L Kosakovsky Pond, Ryan McBride, Charli Worth, Facundo Batista, Devin Sok, IAVI Protocol C Investigators &lt;sup&gt;&amp;&lt;/sup&gt; The IAVI African HIV Research Network, Pascal Poignard, Bryan Briney, Ian A Wilson, Elise Landais, Dennis R Burton' : '27 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">27 more authors</span> </div> <div class="periodical"> <em>PLoS pathogens</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.ppat.1011416" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Vaccination strategies aimed at maturing broadly neutralizing antibodies (bnAbs) from naïve precursors are hindered by unusual features that characterize these Abs, including insertions and deletions (indels). Longitudinal studies of natural HIV infection cases shed light on the complex processes underlying bnAb development and have suggested a role for superinfection as a potential enhancer of neutralization breadth. Here we describe the development of a potent bnAb lineage that was elicited by two founder viruses to inform vaccine design. The V3-glycan targeting bnAb lineage (PC39-1) was isolated from subtype C-infected IAVI Protocol C elite neutralizer, donor PC39, and is defined by the presence of multiple independent insertions in CDRH1 that range from 1-11 amino acids in length. Memory B cell members of this lineage are predominantly atypical in phenotype yet also span the class-switched and antibody-secreting cell compartments. Development of neutralization breadth occurred concomitantly with extensive recombination between founder viruses before each virus separated into two distinct population “arms” that evolved independently to escape the PC39-1 lineage. Ab crystal structures show an extended CDRH1 that can help stabilize the CDRH3. Overall, these findings suggest that early exposure of the humoral system to multiple related Env molecules could promote the induction of bnAbs by focusing Ab responses to conserved epitopes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Perrett2023-lp" class="col-sm-8"> <div class="title">Structural conservation of Lassa virus glycoproteins and recognition by neutralizing antibodies</div> <div class="author"> Hailee R Perrett, Philip J M Brouwer, Jonathan Hurtado, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Maddy L Newby, Lin Liu, Helena Müller-Kräuter, Sarah Müller Aguirre, Judith A Burger, Joey H Bouhuijs, Grace Gibson, Terrence Messmer, John S Schieffelin, Aleksandar Antanasijevic, Geert-Jan Boons, Thomas Strecker, Max Crispin, Rogier W Sanders, Bryan Briney, Andrew B Ward' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>Cell reports</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2023.112524" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Lassa fever is an acute hemorrhagic fever caused by the zoonotic Lassa virus (LASV). The LASV glycoprotein complex (GPC) mediates viral entry and is the sole target for neutralizing antibodies. Immunogen design is complicated by the metastable nature of recombinant GPCs and the antigenic differences among phylogenetically distinct LASV lineages. Despite the sequence diversity of the GPC, structures of most lineages are lacking. We present the development and characterization of prefusion-stabilized, trimeric GPCs of LASV lineages II, V, and VII, revealing structural conservation despite sequence diversity. High-resolution structures and biophysical characterization of the GPC in complex with GP1-A-specific antibodies suggest their neutralization mechanisms. Finally, we present the isolation and characterization of a trimer-preferring neutralizing antibody belonging to the GPC-B competition group with an epitope that spans adjacent protomers and includes the fusion peptide. Our work provides molecular detail information on LASV antigenic diversity and will guide efforts to design pan-LASV vaccines.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Zhou2023-hj" class="col-sm-8"> <div class="title">Broadly neutralizing anti-S2 antibodies protect against all three human betacoronaviruses that cause deadly disease</div> <div class="author"> Panpan Zhou, Ge Song, Hejun Liu, and <span class="more-authors" title="click to view 29 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '29 more authors' ? 'Meng Yuan, Wan-Ting He, Nathan Beutler, Xueyong Zhu, Longping V Tse, David R Martinez, Alexandra Schäfer, Fabio Anzanello, Peter Yong, Linghang Peng, Katharina Dueker, Rami Musharrafieh, Sean Callaghan, Tazio Capozzola, Oliver Limbo, Mara Parren, Elijah Garcia, Stephen A Rawlings, Davey M Smith, David Nemazee, Joseph G Jardine, Yana Safonova, Bryan Briney, Thomas F Rogers, Ian A Wilson, Ralph S Baric, Lisa E Gralinski, Dennis R Burton, Raiees Andrabi' : '29 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">29 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2023.02.005" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Pan-betacoronavirus neutralizing antibodies may hold the key to developing broadly protective vaccines against novel pandemic coronaviruses and to more effectively respond to SARS-CoV-2 variants. The emergence of Omicron and subvariants of SARS-CoV-2 illustrates the limitations of solely targeting the receptor-binding domain (RBD) of the spike (S) protein. Here, we isolated a large panel of broadly neutralizing antibodies (bnAbs) from SARS-CoV-2 recovered-vaccinated donors, which targets a conserved S2 region in the betacoronavirus spike fusion machinery. Select bnAbs showed broad in vivo protection against all three deadly betacoronaviruses, SARS-CoV-1, SARS-CoV-2, and MERS-CoV, which have spilled over into humans in the past two decades. Structural studies of these bnAbs delineated the molecular basis for their broad reactivity and revealed common antibody features targetable by broad vaccination strategies. These bnAbs provide new insights and opportunities for antibody-based interventions and for developing pan-betacoronavirus vaccines.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Cable2023-ye" class="col-sm-8"> <div class="title">Viral immunity: Basic mechanisms and therapeutic applications-a Keystone Symposia report</div> <div class="author"> Jennifer Cable, Siddharth Balachandran, Lisa P Daley-Bauer, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Arjun Rustagi, Ferrin Antony, Justin J Frere, Jamie Strampe, Katherine Kedzierska, Judy L Cannon, Maureen A McGargill, Daniela Weiskopf, Robert C Mettelman, Julia Niessl, Paul G Thomas, Bryan Briney, Sophie A Valkenburg, Jesse D Bloom, Pamela J Bjorkman, Sho Iketani, C Garrett Rappazzo, Chelsea M Crooks, Kali F Crofts, Stefan Pöhlmann, Florian Krammer, Andrea J Sant, Gary J Nabel, Stacey Schultz-Cherry' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Annals of the New York Academy of Sciences</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1111/nyas.14960" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Viruses infect millions of people each year. Both endemic viruses circulating throughout the population as well as novel epidemic and pandemic viruses pose ongoing threats to global public health. Developing more effective tools to address viruses requires not only in-depth knowledge of the virus itself but also of our immune system’s response to infection. On June 29 to July 2, 2022, researchers met for the Keystone symposium “Viral Immunity: Basic Mechanisms and Therapeutic Applications.” This report presents concise summaries from several of the symposium presenters.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Hurtado2023-ay" class="col-sm-8"> <div class="title">Deep repertoire mining uncovers ultra-broad coronavirus neutralizing antibodies targeting multiple spike epitopes</div> <div class="author"> Jonathan Hurtado, Thomas F Rogers, David B Jaffe, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Bruce A Adams, Sandhya Bangaru, Elijah Garcia, Tazio Capozzola, Terrence Messmer, Pragati Sharma, Ge Song, Nathan Beutler, Wanting He, Katharina Dueker, Rami Musharrafieh, Michael J T Stubbington, Dennis R Burton, Raiees Andrabi, Andrew B Ward, Wyatt J McDonnell, Bryan Briney' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.5281/ZENODO.8432479" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Abstract: Development of vaccines and therapeutics that are broadly effective against known and emergent coronaviruses is an urgent priority. We screened the circulating B cell repertoires of COVID-19 survivors and vaccinees to isolate over 9,000 SARS-CoV-2-specific monoclonal Abs (mAbs), providing an expansive view of the SARS-CoV-2-specific Ab repertoire. Among the recovered antibodies was TXG-0078, an NTD-specific neutralizing mAb that recognizes diverse alpha- and beta-coronaviruses. TXG-0078 achieves its exceptional binding breadth while utilizing the same VH1-24 variable gene signature and heavy chain-dominant binding pattern seen in other NTD supersite-specific neutralizing Abs with much narrower specificity. We also report the discovery of CC24.2, a pan-sarbecovirus neutralizing antibody that targets a novel RBD epitope and shows similar neutralization potency against all tested SARS-CoV-2 variants, including BQ.1.1 and XBB.1.5. A cocktail of TXG-0078 and CC24.2 protects in vivo, suggesting potential use in variant-resistant therapeutic Ab cocktails and as templates for pan-coronavirus vaccine design. Datasets: This repository contains the 10x Genomic cellranger outputs (matrix and vdj contig files) as well as complied functional characterization dataset used to generate figures on the publication “Deep repertoire mining uncovers ultra-broad coronavirus neutralizing antibodies targeting multiple spike epitopes”. Post-vaccination samples for donors CC10, CC25, CC31, CC66 were processed in single 10x Genomic reactions. The timepoints samples consist of multiplexing donors CC10, CC25, CC31, CC66 into one 10x Genomic reaction. Similarly, donors CC26, CC42, CC62, CC67 were multiplexed into a single 10x Genomic reaction. Files: feature names.csv - csv file with sort bait/antigen barcode key feature_reference.csv - csv file with cell hash and antigen barcode reference filtered_contig_annotations.csv - High-level annotations of each high-confidence contigs from cell-associated barcodes. This is a subset of all_contig_annotations.csv. filtered_contig.fasta - filtered antibody fasta filtered_matrix.mtx.gz - 10x Genomic matrix file for filtered cells. Contains counts data for feature and gene expression library. raw_matrix.mtx.gz - 10x Genomic matrix file for unfiltered cells. Contains counts data for feature and gene expression library. Code: All code used to generate analysis and figures is available under the MIT license on Github</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2022-fu" class="col-sm-8"> <div class="title">AntiRef: reference clusters of human antibody sequences</div> <div class="author"> <em>Bryan Briney</em> </div> <div class="periodical"> 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.5281/ZENODO.7474336" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Motivation: Biases in the human antibody repertoire result in publicly available antibody sequence datasets containing many duplicate or highly similar sequences. These redundant sequences are a barrier to rapid similarity searches and reduce the efficiency with which these datasets can be used to train statistical or machine learning models of human antibodies. Identity-based clustering provides a solution, however, the extremely large size of available antibody repertoire datasets make such clustering operations computationally intensive and potentially out of reach for many scientists and researchers who would benefit from such data. Results: AntiRef (Antibody Reference Clusters), which is modeled after UniRef, provides clustered datasets of filtered human antibody sequences. Starting from a dataset of  335M unique, full-length, productive human antibody sequences from the Observed Antibody Space repository, several AntiRef cluster sets were generated. Due to the modular nature of recombined antibody genes, the clustering thresholds used by UniRef (100, 90 and 50 percent identity) to cluster general protein sequences are suboptimal for antibody clustering. AntiRef provides reference antibody sequence datasets clustered at a range of relevant identity thresholds: 100, 99, 98, 96, 94, 92 and 90 percent. AntiRef90, which uses the lowest clustering threshold of any AntiRef dataset, is roughly one-third the size of the filtered input dataset and less than half the size of the non-redundant AntiRef100. Datasets: AntiRef comprises a series of datasets, each representing one of several clustering thresholds. AntiRef datasets were generated by a nested clustering procedure similar to UniRef which, proceeding in order of decreasing stringency, clusters the representative sequences from the preceding round of clustering. AntiRef datasets can be found at the following links: AntiRef100: representative sequences resulting from clustering all filtered AntiRef input sequences at 100% identity. AntiRef99: representative sequences resulting from clustering AntiRef100 at 99% identity. AntiRef98: representative sequences resulting from clustering AntiRef99 at 98% identity. AntiRef96: representative sequences resulting from clustering AntiRef98 at 96% identity. AntiRef94: representative sequences resulting from clustering AntiRef96 at 94% identity. AntiRef92: representative sequences resulting from clustering AntiRef94 at 92% identity. AntiRef90: representative sequences resulting from clustering AntiRef92 at 90% identity. Files: The following files are included in the primary AntiRef data repository: antiref_cluster-manifest.csv.gz: A compressed CSV file containing the cluster assignments for every sequence in the AntiRef input dataset. For each AntiRef round, cluster names correspond to the sequence ID of the representative sequence (as determined by MMSeqs2). The nested clustering process conserves cluster names between iterations, meaning the clustering lineage of any sequence can easily be traced across all AntiRef datasets. download_heavy.txt: A plain text file (generated by the Observed Antibody Space) containing the commands necessary to download all antibody heavy chain sequences used to create AntiRef. download_light.txt: A plain text file (generated by the Observed Antibody Space) containing the commands necessary to download all antibody light chain sequences used to create AntiRef. Code: All code used to generate AntiRef (data download, filtering, and clustering) is available under the MIT license on GitHub.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="He2022-wo" class="col-sm-8"> <div class="title">Broadly neutralizing antibodies to SARS-related viruses can be readily induced in rhesus macaques</div> <div class="author"> Wan-Ting He, Meng Yuan, Sean Callaghan, and <span class="more-authors" title="click to view 35 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '35 more authors' ? 'Rami Musharrafieh, Ge Song, Murillo Silva, Nathan Beutler, Wen-Hsin Lee, Peter Yong, Jonathan L Torres, Mariane Melo, Panpan Zhou, Fangzhu Zhao, Xueyong Zhu, Linghang Peng, Deli Huang, Fabio Anzanello, James Ricketts, Mara Parren, Elijah Garcia, Melissa Ferguson, William Rinaldi, Stephen A Rawlings, David Nemazee, Davey M Smith, Bryan Briney, Yana Safonova, Thomas F Rogers, Jennifer M Dan, Zeli Zhang, Daniela Weiskopf, Alessandro Sette, Shane Crotty, Darrell J Irvine, Andrew B Ward, Ian A Wilson, Dennis R Burton, Raiees Andrabi' : '35 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">35 more authors</span> </div> <div class="periodical"> <em>Science translational medicine</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/scitranslmed.abl9605" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>To prepare for future coronavirus (CoV) pandemics, it is desirable to generate vaccines capable of eliciting broadly neutralizing antibody responses to CoVs. Here, we show that immunization of macaques with SARS-CoV-2 spike (S) protein with a two-shot protocol generated potent serum receptor binding domain cross-neutralizing antibody responses to both SARS-CoV-2 and SARS-CoV-1. Furthermore, responses were equally effective against most SARS-CoV-2 variants of concern (VOCs) and some were highly effective against Omicron. This result contrasts with human infection or many two-shot vaccination protocols where responses were typically more SARS-CoV-2 specific and where VOCs were less well neutralized. Structural studies showed that cloned macaque neutralizing antibodies, particularly using a given heavy chain germline gene, recognized a relatively conserved region proximal to the angiotensin converting enzyme 2 receptor binding site (RBS), whereas many frequently elicited human neutralizing antibodies targeted more variable epitopes overlapping the RBS. B cell repertoire differences between humans and macaques appeared to influence the vaccine response. The macaque neutralizing antibodies identified a pan-SARS-related virus epitope region less well targeted by human antibodies that could be exploited in rational vaccine design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Hurtado2022-yo" class="col-sm-8"> <div class="title">Efficient isolation of rare B cells using next-generation antigen barcoding</div> <div class="author"> Jonathan Hurtado, Claudia Flynn, Jeong Hyun Lee, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Eugenia C Salcedo, Christopher A Cottrell, Patrick D Skog, Dennis R Burton, David Nemazee, William R Schief, Elise Landais, Devin Sok, Bryan Briney' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Frontiers in cellular and infection microbiology</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fcimb.2022.962945" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The ability to efficiently isolate antigen-specific B cells in high throughput will greatly accelerate the discovery of therapeutic monoclonal antibodies (mAbs) and catalyze rational vaccine development. Traditional mAb discovery is a costly and labor-intensive process, although recent advances in single-cell genomics using emulsion microfluidics allow simultaneous processing of thousands of individual cells. Here we present a streamlined method for isolation and analysis of large numbers of antigen-specific B cells, including next generation antigen barcoding and an integrated computational framework for B cell multi-omics. We demonstrate the power of this approach by recovering thousands of antigen-specific mAbs, including the efficient isolation of extremely rare precursors of VRC01-class and IOMA-class broadly neutralizing HIV mAbs.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Ben-Othman2020-qf" class="col-sm-8"> <div class="title">Systems biology methods applied to blood and tissue for a comprehensive analysis of immune response to hepatitis B vaccine in adults</div> <div class="author"> Rym Ben-Othman, Bing Cai, Aaron C Liu, and <span class="more-authors" title="click to view 43 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '43 more authors' ? 'Natallia Varankovich, Daniel He, Travis M Blimkie, Amy H Lee, Erin E Gill, Mark Novotny, Brian Aevermann, Sibyl Drissler, Casey P Shannon, Sarah McCann, Kim Marty, Gordean Bjornson, Rachel D Edgar, David Tse Shen Lin, Nicole Gladish, Julia Maclsaac, Nelly Amenyogbe, Queenie Chan, Alba Llibre, Joyce Collin, Elise Landais, Khoa Le, Samantha M Reiss, Wayne C Koff, Colin Havenar-Daughton, Manraj Heran, Bippan Sangha, David Walt, Mel Krajden, Shane Crotty, Devin Sok, Bryan Briney, Dennis R Burton, Darragh Duffy, Leonard J Foster, William W Mohn, Michael S Kobor, Scott J Tebbutt, Ryan R Brinkman, Richard H Scheuermann, Robert E W Hancock, Tobias R Kollmann, Manish Sadarangani' : '43 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">43 more authors</span> </div> <div class="periodical"> <em>Frontiers in immunology</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fimmu.2020.580373" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Conventional vaccine design has been based on trial-and-error approaches, which have been generally successful. However, there have been some major failures in vaccine development and we still do not have highly effective licensed vaccines for tuberculosis, HIV, respiratory syncytial virus, and other major infections of global significance. Approaches at rational vaccine design have been limited by our understanding of the immune response to vaccination at the molecular level. Tools now exist to undertake in-depth analysis using systems biology approaches, but to be fully realized, studies are required in humans with intensive blood and tissue sampling. Methods that support this intensive sampling need to be developed and validated as feasible. To this end, we describe here a detailed approach that was applied in a study of 15 healthy adults, who were immunized with hepatitis B vaccine. Sampling included  350 mL of blood, 12 microbiome samples, and lymph node fine needle aspirates obtained over a  7-month period, enabling comprehensive analysis of the immune response at the molecular level, including single cell and tissue sample analysis. Samples were collected for analysis of immune phenotyping, whole blood and single cell gene expression, proteomics, lipidomics, epigenetics, whole blood response to key immune stimuli, cytokine responses, in vitro T cell responses, antibody repertoire analysis and the microbiome. Data integration was undertaken using different approaches-NetworkAnalyst and DIABLO. Our results demonstrate that such intensive sampling studies are feasible in healthy adults, and data integration tools exist to analyze the vast amount of data generated from a multi-omics systems biology approach. This will provide the basis for a better understanding of vaccine-induced immunity and accelerate future rational vaccine design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Zhao2020-ee" class="col-sm-8"> <div class="title">Mapping neutralizing antibody Epitope specificities to an HIV Env trimer in immunized and in infected rhesus macaques</div> <div class="author"> Fangzhu Zhao, Collin Joyce, Alison Burns, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Bartek Nogal, Christopher A Cottrell, Alejandra Ramos, Trevor Biddle, Matthias Pauthner, Rebecca Nedellec, Huma Qureshi, Rosemarie Mason, Elise Landais, Bryan Briney, Andrew B Ward, Dennis R Burton, Devin Sok' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>Cell reports</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2020.108122" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>BG505 SOSIP is a well-characterized near-native recombinant HIV Envelope (Env) trimer that holds promise as part of a sequential HIV immunogen regimen to induce broadly neutralizing antibodies (bnAbs). Rhesus macaques are considered the most appropriate pre-clinical animal model for monitoring antibody (Ab) responses. Accordingly, we report here the isolation of 45 BG505 autologous neutralizing antibodies (nAbs) with multiple specificities from SOSIP-immunized and BG505 SHIV-infected rhesus macaques. We associate the most potent neutralization with two epitopes: the C3/V5 and V1/V3 regions. We show that all of the nAbs bind in close proximity to known bnAb epitopes and might therefore sterically hinder elicitation of bnAbs. We also identify a “public clonotype” that targets the immunodominant C3/V5 nAb epitope, which suggests that common antibody rearrangements might help determine humoral responses to Env immunogens. The results highlight important considerations for vaccine design in anticipation of results of the BG505 SOSIP trimer in clinical trials.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Rogers2020-lc" class="col-sm-8"> <div class="title">Isolation of potent SARS-CoV-2 neutralizing antibodies and protection from disease in a small animal model</div> <div class="author"> Thomas F Rogers, Fangzhu Zhao, Deli Huang, and <span class="more-authors" title="click to view 31 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '31 more authors' ? 'Nathan Beutler, Alison Burns, Wan-Ting He, Oliver Limbo, Chloe Smith, Ge Song, Jordan Woehl, Linlin Yang, Robert K Abbott, Sean Callaghan, Elijah Garcia, Jonathan Hurtado, Mara Parren, Linghang Peng, Sydney Ramirez, James Ricketts, Michael J Ricciardi, Stephen A Rawlings, Nicholas C Wu, Meng Yuan, Davey M Smith, David Nemazee, John R Teijaro, James E Voss, Ian A Wilson, Raiees Andrabi, Bryan Briney, Elise Landais, Devin Sok, Joseph G Jardine, Dennis R Burton' : '31 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">31 more authors</span> </div> <div class="periodical"> <em>Science (New York, N.Y.)</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.abc7520" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Countermeasures to prevent and treat coronavirus disease 2019 (COVID-19) are a global health priority. We enrolled a cohort of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-recovered participants, developed neutralization assays to investigate antibody responses, adapted our high-throughput antibody generation pipeline to rapidly screen more than 1800 antibodies, and established an animal model to test protection. We isolated potent neutralizing antibodies (nAbs) to two epitopes on the receptor binding domain (RBD) and to distinct non-RBD epitopes on the spike (S) protein. As indicated by maintained weight and low lung viral titers in treated animals, the passive transfer of a nAb provides protection against disease in high-dose SARS-CoV-2 challenge in Syrian hamsters. The study suggests a role for nAbs in prophylaxis, and potentially therapy, of COVID-19. The nAbs also define protective epitopes to guide vaccine design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Rogers2020-ta" class="col-sm-8"> <div class="title">Rapid isolation of potent SARS-CoV-2 neutralizing antibodies and protection in a small animal model</div> <div class="author"> Thomas F Rogers, Fangzhu Zhao, Deli Huang, and <span class="more-authors" title="click to view 27 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '27 more authors' ? 'Nathan Beutler, Alison Burns, Wan-Ting He, Oliver Limbo, Chloe Smith, Ge Song, Jordan Woehl, Linlin Yang, Robert K Abbott, Sean Callaghan, Elijah Garcia, Jonathan Hurtado, Mara Parren, Linghang Peng, James Ricketts, Michael J Ricciardi, Stephen A Rawlings, Davey M Smith, David Nemazee, John R Teijaro, James E Voss, Raiees Andrabi, Bryan Briney, Elise Landais, Devin Sok, Joseph G Jardine, Dennis R Burton' : '27 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">27 more authors</span> </div> <div class="periodical"> <em>bioRxiv</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/2020.05.11.088674" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The development of countermeasures to prevent and treat COVID-19 is a global health priority. In under 7 weeks, we enrolled a cohort of SARS-CoV-2-recovered participants, developed neutralization assays to interrogate serum and monoclonal antibody responses, adapted our high throughput antibody isolation, production and characterization pipeline to rapidly screen over 1000 antigen-specific antibodies, and established an animal model to test protection. We report multiple highly potent neutralizing antibodies (nAbs) and show that passive transfer of a nAb provides protection against high-dose SARS-CoV-2 challenge in Syrian hamsters. The study suggests a role for nAbs in prophylaxis, and potentially therapy, of COVID-19. The nAbs define protective epitopes to guide vaccine design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Smakaj2020-ei" class="col-sm-8"> <div class="title">Benchmarking immunoinformatic tools for the analysis of antibody repertoire sequences</div> <div class="author"> Erand Smakaj, Lmar Babrak, Mats Ohlin, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Mikhail Shugay, Bryan Briney, Deniz Tosoni, Christopher Galli, Vendi Grobelsek, Igor D’Angelo, Branden Olson, Sai Reddy, Victor Greiff, Johannes Trück, Susanna Marquez, William Lees, Enkelejda Miho' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>Bioinformatics (Oxford, England)</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/bioinformatics/btz845" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>SUMMARY: Antibody repertoires reveal insights into the biology of the adaptive immune system and empower diagnostics and therapeutics. There are currently multiple tools available for the annotation of antibody sequences. All downstream analyses such as choosing lead drug candidates depend on the correct annotation of these sequences; however, a thorough comparison of the performance of these tools has not been investigated. Here, we benchmark the performance of commonly used immunoinformatic tools, i.e. IMGT/HighV-QUEST, IgBLAST and MiXCR, in terms of reproducibility of annotation output, accuracy and speed using simulated and experimental high-throughput sequencing datasets.We analyzed changes in IMGT reference germline database in the last 10 years in order to assess the reproducibility of the annotation output. We found that only 73/183 (40%) V, D and J human genes were shared between the reference germline sets used by the tools. We found that the annotation results differed between tools. In terms of alignment accuracy, MiXCR had the highest average frequency of gene mishits, 0.02 mishit frequency and IgBLAST the lowest, 0.004 mishit frequency. Reproducibility in the output of complementarity determining three regions (CDR3 amino acids) ranged from 4.3% to 77.6% with preprocessed data. In addition, run time of the tools was assessed: MiXCR was the fastest tool for number of sequences processed per unit of time. These results indicate that immunoinformatic analyses greatly depend on the choice of bioinformatics tool. Our results support informed decision-making to immunoinformaticians based on repertoire composition and sequencing platforms. AVAILABILITY AND IMPLEMENTATION: All tools utilized in the paper are free for academic use. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Joyce2020-vo" class="col-sm-8"> <div class="title">Comparisons of the antibody repertoires of a humanized rodent and humans by high throughput sequencing</div> <div class="author"> Collin Joyce, Dennis R Burton, and <em>Bryan Briney</em> </div> <div class="periodical"> <em>Scientific reports</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41598-020-57764-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The humanization of animal model immune systems by genetic engineering has shown great promise for antibody discovery, tolerance studies and for the evaluation of vaccines. Assessment of the baseline antibody repertoires of unimmunized model animals will be useful as a benchmark for future immunization experiments. We characterized the heavy chain and kappa light chain antibody repertoires of a model animal, the OmniRat, by high throughput antibody sequencing and made use of two novel datasets for comparison to human repertoires. Intra-animal and inter-animal repertoire comparisons reveal a high level of conservation in antibody diversity between the lymph node and spleen and between members of the species. Multiple differences were found in both the heavy and kappa chain repertoires between OmniRats and humans including gene segment usage, CDR3 length distributions, class switch recombination, somatic hypermutation levels and in features of V(D)J recombination. The Inference and Generation of Repertoires (IGoR) software tool was used to model recombination in VH regions which allowed for the quantification of some of these differences. Diversity estimates of the OmniRat heavy chain repertoires almost reached that of humans, around two orders of magnitude less. Despite variation between the species repertoires, a high frequency of OmniRat clonotypes were also found in the human repertoire. These data give insights into the development and selection of humanized animal antibodies and provide actionable information for use in vaccine studies.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Steichen2019-bc" class="col-sm-8"> <div class="title">A generalized HIV vaccine design strategy for priming of broadly neutralizing antibody responses</div> <div class="author"> Jon M Steichen, Ying-Cing Lin, Colin Havenar-Daughton, and <span class="more-authors" title="click to view 28 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '28 more authors' ? 'Simone Pecetta, Gabriel Ozorowski, Jordan R Willis, Laura Toy, Devin Sok, Alessia Liguori, Sven Kratochvil, Jonathan L Torres, Oleksandr Kalyuzhniy, Eleonora Melzi, Daniel W Kulp, Sebastian Raemisch, Xiaozhen Hu, Steffen M Bernard, Erik Georgeson, Nicole Phelps, Yumiko Adachi, Michael Kubitz, Elise Landais, Jeffrey Umotoy, Amanda Robinson, Bryan Briney, Ian A Wilson, Dennis R Burton, Andrew B Ward, Shane Crotty, Facundo D Batista, William R Schief' : '28 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">28 more authors</span> </div> <div class="periodical"> <em>Science (New York, N.Y.)</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.aax4380" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Vaccine induction of broadly neutralizing antibodies (bnAbs) to HIV remains a major challenge. Germline-targeting immunogens hold promise for initiating the induction of certain bnAb classes; yet for most bnAbs, a strong dependence on antibody heavy chain complementarity-determining region 3 (HCDR3) is a major barrier. Exploiting ultradeep human antibody sequencing data, we identified a diverse set of potential antibody precursors for a bnAb with dominant HCDR3 contacts. We then developed HIV envelope trimer-based immunogens that primed responses from rare bnAb-precursor B cells in a mouse model and bound a range of potential bnAb-precursor human naïve B cells in ex vivo screens. Our repertoire-guided germline-targeting approach provides a framework for priming the induction of many HIV bnAbs and could be applied to most HCDR3-dominant antibodies from other pathogens.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Havenar-Daughton2019-eg" class="col-sm-8"> <div class="title">Rapid germinal center and antibody responses in non-human primates after a single nanoparticle vaccine immunization</div> <div class="author"> Colin Havenar-Daughton, Diane G Carnathan, Archana V Boopathy, and <span class="more-authors" title="click to view 17 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '17 more authors' ? 'Amit A Upadhyay, Ben Murrell, Samantha M Reiss, Chiamaka A Enemuo, Etse H Gebru, Yury Choe, Pallavi Dhadvai, Federico Viviano, Kirti Kaushik, Jinal N Bhiman, Bryan Briney, Dennis R Burton, Steven E Bosinger, William R Schief, Darrell J Irvine, Guido Silvestri, Shane Crotty' : '17 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">17 more authors</span> </div> <div class="periodical"> <em>Cell reports</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2019.10.008" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The first immunization in a protein prime-boost vaccination is likely to be critical for how the immune response unfolds. Using fine needle aspirates (FNAs) of draining lymph nodes (LNs), we tracked the kinetics of the primary immune response in rhesus monkeys immunized intramuscularly (IM) or subcutaneously (s.c.) with an eOD-GT8 60-mer nanoparticle immunogen to facilitate clinical trial design. Significant numbers of germinal center B (BGC) cells and antigen-specific CD4 T cells were detectable in the draining LN as early as 7 days post-immunization and peaked near day 21. Strikingly, s.c. immunization results in 10-fold larger antigen-specific BGC cell responses compared to IM immunization. Lymphatic drainage studies revealed that s.c. immunization resulted in faster and more consistent axillary LN drainage than IM immunization. These data indicate robust antigen-specific germinal center responses can occur rapidly to a single immunization with a nanoparticle immunogen and vaccine drainage substantially impacts immune responses in local LNs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Blasco2019-bu" class="col-sm-8"> <div class="title">Advancing computational biology and bioinformatics research through open innovation competitions</div> <div class="author"> Andrea Blasco, Michael G Endres, Rinat A Sergeev, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Anup Jonchhe, N J Maximilian Macaluso, Rajiv Narayan, Ted Natoli, Jin H Paik, Bryan Briney, Chunlei Wu, Andrew I Su, Aravind Subramanian, Karim R Lakhani' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>PloS one</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pone.0222165" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Open data science and algorithm development competitions offer a unique avenue for rapid discovery of better computational strategies. We highlight three examples in computational biology and bioinformatics research in which the use of competitions has yielded significant performance gains over established algorithms. These include algorithms for antibody clustering, imputing gene expression data, and querying the Connectivity Map (CMap). Performance gains are evaluated quantitatively using realistic, albeit sanitized, data sets. The solutions produced through these competitions are then examined with respect to their utility and the prospects for implementation in the field. We present the decision process and competition design considerations that lead to these successful outcomes as a model for researchers who want to use competitions and non-domain crowds as collaborators to further their research.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Umotoy2019-ma" class="col-sm-8"> <div class="title">Rapid and focused maturation of a VRC01-class HIV broadly neutralizing antibody lineage involves both binding and accommodation of the N276-glycan</div> <div class="author"> Jeffrey Umotoy, Bernard S Bagaya, Collin Joyce, and <span class="more-authors" title="click to view 22 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '22 more authors' ? 'Torben Schiffner, Sergey Menis, Karen L Saye-Francisco, Trevor Biddle, Sanjay Mohan, Thomas Vollbrecht, Oleksander Kalyuzhniy, Sharon Madzorera, Dale Kitchin, Bronwen Lambson, Molati Nonyane, William Kilembe, IAVI Protocol C Investigators, IAVI African HIV Research Network, Pascal Poignard, William R Schief, Dennis R Burton, Ben Murrell, Penny L Moore, Bryan Briney, Devin Sok, Elise Landais' : '22 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">22 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2019.06.004" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The VH1-2 restricted VRC01-class of antibodies targeting the HIV envelope CD4 binding site are a major focus of HIV vaccine strategies. However, a detailed analysis of VRC01-class antibody development has been limited by the rare nature of these responses during natural infection and the lack of longitudinal sampling of such responses. To inform vaccine strategies, we mapped the development of a VRC01-class antibody lineage (PCIN63) in the subtype C infected IAVI Protocol C neutralizer PC063. PCIN63 monoclonal antibodies had the hallmark VRC01-class features and demonstrated neutralization breadth similar to the prototype VRC01 antibody, but were 2- to 3-fold less mutated. Maturation occurred rapidly within ∼24 months of emergence of the lineage and somatic hypermutations accumulated at key contact residues. This longitudinal study of broadly neutralizing VRC01-class antibody lineage reveals early binding to the N276-glycan during affinity maturation, which may have implications for vaccine design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2019-qy" class="col-sm-8"> <div class="title">Commonality despite exceptional diversity in the baseline human antibody repertoire</div> <div class="author"> <em>Bryan Briney</em>, Anne Inderbitzin, Collin Joyce, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Dennis R Burton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Nature</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41586-019-0879-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>In principle, humans can produce an antibody response to any non-self-antigen molecule in the appropriate context. This flexibility is achieved by the presence of a large repertoire of naive antibodies, the diversity of which is expanded by somatic hypermutation following antigen exposure1. The diversity of the naive antibody repertoire in humans is estimated to be at least 1012 unique antibodies2. Because the number of peripheral blood B cells in a healthy adult human is on the order of 5 × 109, the circulating B cell population samples only a small fraction of this diversity. Full-scale analyses of human antibody repertoires have been prohibitively difficult, primarily owing to their massive size. The amount of information encoded by all of the rearranged antibody and T cell receptor genes in one person-the ’genome’ of the adaptive immune system-exceeds the size of the human genome by more than four orders of magnitude. Furthermore, because much of the B lymphocyte population is localized in organs or tissues that cannot be comprehensively sampled from living subjects, human repertoire studies have focused on circulating B cells3. Here we examine the circulating B cell populations of ten human subjects and present what is, to our knowledge, the largest single collection of adaptive immune receptor sequences described to date, comprising almost 3 billion antibody heavy-chain sequences. This dataset enables genetic study of the baseline human antibody repertoire at an unprecedented depth and granularity, which reveals largely unique repertoires for each individual studied, a subpopulation of universally shared antibody clonotypes, and an exceptional overall diversity of the antibody repertoire.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Voss2019-tl" class="col-sm-8"> <div class="title">Reprogramming the antigen specificity of B cells using genome-editing technologies</div> <div class="author"> James E Voss, Alicia Gonzalez-Martin, Raiees Andrabi, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Roberta P Fuller, Ben Murrell, Laura E McCoy, Katelyn Porter, Deli Huang, Wenjuan Li, Devin Sok, Khoa Le, Bryan Briney, Morgan Chateau, Geoffrey Rogers, Lars Hangartner, Ann J Feeney, David Nemazee, Paula Cannon, Dennis R Burton' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>eLife</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.42995" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We have developed a method to introduce novel paratopes into the human antibody repertoire by modifying the immunoglobulin (Ig) genes of mature B cells directly using genome editing technologies. We used CRISPR-Cas9 in a homology directed repair strategy, to replace the heavy chain (HC) variable region in B cell lines with that from an HIV broadly neutralizing antibody (bnAb), PG9. Our strategy is designed to function in cells that have undergone VDJ recombination using any combination of variable (V), diversity (D) and joining (J) genes. The modified locus expresses PG9 HC which pairs with native light chains (LCs) resulting in the cell surface expression of HIV specific B cell receptors (BCRs). Endogenous activation-induced cytidine deaminase (AID) in engineered cells allowed for Ig class switching and generated BCR variants with improved HIV neutralizing activity. Thus, BCRs engineered in this way retain the genetic flexibility normally required for affinity maturation during adaptive immune responses. Peripheral blood derived primary B cells from three different donors were edited using this strategy. Engineered cells could bind the PG9 epitope and sequenced mRNA showed PG9 HC transcribed as several different isotypes after culture with CD40 ligand and IL-4.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2018-zx" class="col-sm-8"> <div class="title">Massively scalable genetic analysis of antibody repertoires</div> <div class="author"> <em>Bryan Briney</em> and Dennis R Burton </div> <div class="periodical"> <em>bioRxiv</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1101/447813" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>AbstractWith technical breakthroughs in the throughput and read-length of next-generation sequencing platforms, antibody repertoire sequencing is becoming an increasingly important tool for detailed characterization of the immune response. There is a need for open, scalable software for the genetic analysis of repertoire-scale antibody sequence data. To address this gap, we have developed the ab[x] package of software tools. There are three core components of the ab[x] toolkit, all of which are freely available: abcloud (github.com/briney/abcloud) for deployment and management of computational resources on Amazon’s Elastic Compute Cloud; abstar (github.com/briney/abstar) for pre-processing, germline gene assignment and primary annotation of antibody sequence data; and abutils (github.com/briney/abutils), which provides utilities for interactive downstream analysis of antibody repertoire data.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Andrabi2017-ns" class="col-sm-8"> <div class="title">Glycans function as anchors for antibodies and help drive HIV broadly neutralizing antibody development</div> <div class="author"> Raiees Andrabi, Ching-Yao Su, Chi-Hui Liang, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Sachin S Shivatare, Bryan Briney, James E Voss, Salar Khan Nawazi, Chung-Yi Wu, Chi-Huey Wong, Dennis R Burton' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2017.10.012" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>SummaryApex broadly neutralizing HIV antibodies (bnAbs) recognize glycans and protein surface close to the 3-fold axis of the envelope (Env) trimer and are among the most potent and broad Abs described. The evolution of apex bnAbs from one donor (CAP256) has been studied in detail and many Abs at different stages of maturation have been described. Using diverse engineering tools, we investigated the involvement of glycan recognition in the development of the CAP256.VRC26 Ab lineage. We found that sialic acid-bearing glycans were recognized by germline-encoded and somatically mutated residues on the Ab heavy chain. This recognition provided an “anchor” for the Abs as the core protein epitope varies, prevented complete neutralization escape, and eventually led to broadening of the response. These findings illustrate how glycan-specific maturation enables a human Ab to cope with pathogen escape mechanisms and will aid in optimization of immunization strategies to induce V2 apex bnAb responses.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Landais2017-iw" class="col-sm-8"> <div class="title">HIV envelope glycoform heterogeneity and localized diversity govern the initiation and maturation of a V2 apex broadly neutralizing antibody lineage</div> <div class="author"> Elise Landais, Ben Murrell, <em>Bryan Briney</em>, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Sasha Murrell, Kimmo Rantalainen, Zachary T Berndsen, Alejandra Ramos, Lalinda Wickramasinghe, Melissa Laird Smith, Kemal Eren, Natalia Val, Mengyu Wu, Audrey Cappelletti, Jeffrey Umotoy, Yolanda Lie, Terri Wrin, Paul Algate, Po-Ying Chan-Hui, Etienne Karita, IAVI Protocol C Investigators, IAVI African HIV Research Network, Andrew B Ward, Ian A Wilson, Dennis R Burton, Davey Smith, Sergei L Kosakovsky Pond, Pascal Poignard' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2017.11.002" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Understanding how broadly neutralizing antibodies (bnAbs) to HIV envelope (Env) develop during natural infection can help guide the rational design of an HIV vaccine. Here, we described a bnAb lineage targeting the Env V2 apex and the Ab-Env co-evolution that led to development of neutralization breadth. The lineage Abs bore an anionic heavy chain complementarity-determining region 3 (CDRH3) of 25 amino acids, among the shortest known for this class of Abs, and achieved breadth with only 10% nucleotide somatic hypermutation and no insertions or deletions. The data suggested a role for Env glycoform heterogeneity in the activation of the lineage germline B cell. Finally, we showed that localized diversity at key V2 epitope residues drove bnAb maturation toward breadth, mirroring the Env evolution pattern described for another donor who developed V2-apex targeting bnAbs. Overall, these findings suggest potential strategies for vaccine approaches based on germline-targeting and serial immunogen design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Magnani2017-ax" class="col-sm-8"> <div class="title">Neutralizing human monoclonal antibodies prevent Zika virus infection in macaques</div> <div class="author"> Diogo M Magnani, Thomas F Rogers, Nathan Beutler, and <span class="more-authors" title="click to view 27 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '27 more authors' ? 'Michael J Ricciardi, Varian K Bailey, Lucas Gonzalez-Nieto, Bryan Briney, Devin Sok, Khoa Le, Alexander Strubel, Martin J Gutman, Núria Pedreño-Lopez, Nathan D Grubaugh, Cassia G T Silveira, Helen S Maxwell, Aline Domingues, Mauricio A Martins, David E Lee, Erica E Okwuazi, Sherrie Jean, Elizabeth A Strobert, Ann Chahroudi, Guido Silvestri, Thomas H Vanderford, Esper G Kallas, Ronald C Desrosiers, Myrna C Bonaldo, Stephen S Whitehead, Dennis R Burton, David I Watkins' : '27 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">27 more authors</span> </div> <div class="periodical"> <em>Science translational medicine</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/scitranslmed.aan8184" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Therapies to prevent maternal Zika virus (ZIKV) infection and its subsequent fetal developmental complications are urgently required. We isolated three potent ZIKV-neutralizing monoclonal antibodies (nmAbs) from the plasmablasts of a ZIKV-infected patient-SMZAb1, SMZAb2, and SMZAb5-directed against two different domains of the virus. We engineered these nmAbs with Fc LALA mutations that abrogate Fcγ receptor binding, thus eliminating potential therapy-mediated antibody-dependent enhancement. We administered a cocktail of these three nmAbs to nonhuman primates 1 day before challenge with ZIKV and demonstrated that the nmAbs completely prevented viremia in serum after challenge. Given that numerous antibodies have exceptional safety profiles in humans, the cocktail described here could be rapidly developed to protect uninfected pregnant women and their fetuses.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Rogers2017-tx" class="col-sm-8"> <div class="title">Zika virus activates de novo and cross-reactive memory B cell responses in dengue-experienced donors</div> <div class="author"> Thomas F Rogers, Eileen C Goodwin, <em>Bryan Briney</em>, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Devin Sok, Nathan Beutler, Alexander Strubel, Rebecca Nedellec, Khoa Le, Michael E Brown, Dennis R Burton, Laura M Walker' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">8 more authors</span> </div> <div class="periodical"> <em>Science immunology</em>, 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/sciimmunol.aan6809" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Zika virus (ZIKV) shares a high degree of homology with dengue virus (DENV), suggesting that preexisting immunity to DENV could affect immune responses to ZIKV. We have tracked the evolution of ZIKV-induced B cell responses in three DENV-experienced donors. The acute antibody (plasmablast) responses were characterized by relatively high somatic hypermutation and a bias toward DENV binding and neutralization, implying the early activation of DENV clones. A DENV-naïve donor in contrast showed a classical primary plasmablast response. Five months after infection, the DENV-experienced donors developed potent type-specific ZIKV neutralizing antibody responses in addition to DENV cross-reactive responses. Because cross-reactive responses were poorly neutralizing and associated with enhanced ZIKV infection in vitro, preexisting DENV immunity could negatively affect protective antibody responses to ZIKV. The observed effects are epitope-dependent, suggesting that a ZIKV vaccine should be carefully designed for DENV-seropositive populations.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Havenar-Daughton2016-cz" class="col-sm-8"> <div class="title">Direct probing of germinal center responses reveals immunological features and bottlenecks for neutralizing antibody responses to HIV Env trimer</div> <div class="author"> Colin Havenar-Daughton, Diane G Carnathan, Alba Peña, and <span class="more-authors" title="click to view 25 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '25 more authors' ? 'Matthias Pauthner, Bryan Briney, Samantha M Reiss, Jennifer S Wood, Kirti Kaushik, Marit J Gils, Sandy L Rosales, Patricia Woude, Michela Locci, Khoa M Le, Steven W Taeye, Devin Sok, Ata Ur Rasheed Mohammed, Jessica Huang, Sanjeev Gumber, Anapatricia Garcia, Sudhir P Kasturi, Bali Pulendran, John P Moore, Rafi Ahmed, Grégory Seumois, Dennis R Burton, Rogier W Sanders, Guido Silvestri, Shane Crotty' : '25 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">25 more authors</span> </div> <div class="periodical"> <em>Cell reports</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2016.10.085" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Generating tier 2 HIV-neutralizing antibody (nAb) responses by immunization remains a challenging problem, and the immunological barriers to induction of such responses with Env immunogens remain unclear. Here, some rhesus monkeys developed autologous tier 2 nAbs upon HIV Env trimer immunization (SOSIP.v5.2) whereas others did not. This was not because HIV Env trimers were immunologically silent because all monkeys made similar ELISA-binding antibody responses; the key difference was nAb versus non-nAb responses. We explored the immunological barriers to HIV nAb responses by combining a suite of techniques, including longitudinal lymph node fine needle aspirates. Unexpectedly, nAb development best correlated with booster immunization GC B cell magnitude and Tfh characteristics of the Env-specific CD4 T cells. Notably, these factors distinguished between successful and unsuccessful antibody responses because GC B cell frequencies and stoichiometry to GC Tfh cells correlated with nAb development, but did not correlate with total Env Ab binding titers.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sok2016-fx" class="col-sm-8"> <div class="title">Priming HIV-1 broadly neutralizing antibody precursors in human Ig loci transgenic mice</div> <div class="author"> Devin Sok, <em>Bryan Briney</em>, Joseph G Jardine, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Daniel W Kulp, Sergey Menis, Matthias Pauthner, Andrew Wood, E-Chiang Lee, Khoa M Le, Meaghan Jones, Alejandra Ramos, Oleksandr Kalyuzhniy, Yumiko Adachi, Michael Kubitz, Skye MacPherson, Allan Bradley, Glenn A Friedrich, William R Schief, Dennis R Burton' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>Science (New York, N.Y.)</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.aah3945" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A major obstacle to a broadly neutralizing antibody (bnAb)-based HIV vaccine is the activation of appropriate B cell precursors. Germline-targeting immunogens must be capable of priming rare bnAb precursors in the physiological setting. We tested the ability of the VRC01-class bnAb germline-targeting immunogen eOD-GT8 60mer (60-subunit self-assembling nanoparticle) to activate appropriate precursors in mice transgenic for human immunoglobulin (Ig) loci. Despite an average frequency of, at most, about one VRC01-class precursor per mouse, we found that at least 29% of singly immunized mice produced a VRC01-class memory response, suggesting that priming generally succeeded when at least one precursor was present. The results demonstrate the feasibility of using germline targeting to prime specific and exceedingly rare bnAb-precursor B cells within a humanlike repertoire.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Steichen2016-fm" class="col-sm-8"> <div class="title">HIV vaccine design to target germline precursors of glycan-dependent broadly neutralizing antibodies</div> <div class="author"> Jon M Steichen, Daniel W Kulp, Talar Tokatlian, and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Amelia Escolano, Pia Dosenovic, Robyn L Stanfield, Laura E McCoy, Gabriel Ozorowski, Xiaozhen Hu, Oleksandr Kalyuzhniy, Bryan Briney, Torben Schiffner, Fernando Garces, Natalia T Freund, Alexander D Gitlin, Sergey Menis, Erik Georgeson, Michael Kubitz, Yumiko Adachi, Meaghan Jones, Andrew A Mutafyan, Dong Soo Yun, Christian T Mayer, Andrew B Ward, Dennis R Burton, Ian A Wilson, Darrell J Irvine, Michel C Nussenzweig, William R Schief' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2016.08.016" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Broadly neutralizing antibodies (bnAbs) against the N332 supersite of the HIV envelope (Env) trimer are the most common bnAbs induced during infection, making them promising leads for vaccine design. Wild-type Env glycoproteins lack detectable affinity for supersite-bnAb germline precursors and are therefore unsuitable immunogens to prime supersite-bnAb responses. We employed mammalian cell surface display to design stabilized Env trimers with affinity for germline-reverted precursors of PGT121-class supersite bnAbs. The trimers maintained native-like antigenicity and structure, activated PGT121 inferred-germline B cells ex vivo when multimerized on liposomes, and primed PGT121-like responses in PGT121 inferred-germline knockin mice. Design intermediates have levels of epitope modification between wild-type and germline-targeting trimers; their mutation gradient suggests sequential immunization to induce bnAbs, in which the germline-targeting prime is followed by progressively less-mutated design intermediates and, lastly, with native trimers. The vaccine design strategies described could be utilized to target other epitopes on HIV or other pathogens.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2016-ap" class="col-sm-8"> <div class="title">Tailored immunogens direct affinity maturation toward HIV neutralizing antibodies</div> <div class="author"> <em>Bryan Briney</em>, Devin Sok, Joseph G Jardine, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Daniel W Kulp, Patrick Skog, Sergey Menis, Ronald Jacak, Oleksandr Kalyuzhniy, Natalia Val, Fabian Sesterhenn, Khoa M Le, Alejandra Ramos, Meaghan Jones, Karen L Saye-Francisco, Tanya R Blane, Skye Spencer, Erik Georgeson, Xiaozhen Hu, Gabriel Ozorowski, Yumiko Adachi, Michael Kubitz, Anita Sarkar, Ian A Wilson, Andrew B Ward, David Nemazee, Dennis R Burton, William R Schief' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Cell</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.cell.2016.08.005" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Induction of broadly neutralizing antibodies (bnAbs) is a primary goal of HIV vaccine development. VRC01-class bnAbs are important vaccine leads because their precursor B cells targeted by an engineered priming immunogen are relatively common among humans. This priming immunogen has demonstrated the ability to initiate a bnAb response in animal models, but recall and maturation toward bnAb development has not been shown. Here, we report the development of boosting immunogens designed to guide the genetic and functional maturation of previously primed VRC01-class precursors. Boosting a transgenic mouse model expressing germline VRC01 heavy chains produced broad neutralization of near-native isolates (N276A) and weak neutralization of fully native HIV. Functional and genetic characteristics indicate that the boosted mAbs are consistent with partially mature VRC01-class antibodies and place them on a maturation trajectory that leads toward mature VRC01-class bnAbs. The results show how reductionist sequential immunization can guide maturation of HIV bnAb responses.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="McCoy2016-bb" class="col-sm-8"> <div class="title">Holes in the glycan shield of the native HIV Envelope are a target of trimer-elicited neutralizing antibodies</div> <div class="author"> Laura E McCoy, Marit J Gils, Gabriel Ozorowski, and <span class="more-authors" title="click to view 16 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '16 more authors' ? 'Terrence Messmer, Bryan Briney, James E Voss, Daniel W Kulp, Matthew S Macauley, Devin Sok, Matthias Pauthner, Sergey Menis, Christopher A Cottrell, Jonathan L Torres, Jessica Hsueh, William R Schief, Ian A Wilson, Andrew B Ward, Rogier W Sanders, Dennis R Burton' : '16 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">16 more authors</span> </div> <div class="periodical"> <em>Cell reports</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.celrep.2016.07.074" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A major advance in the search for an HIV vaccine has been the development of a near-native Envelope trimer (BG505 SOSIP.664) that can induce robust autologous Tier 2 neutralization. Here, potently neutralizing monoclonal antibodies (nAbs) from rabbits immunized with BG505 SOSIP.664 are shown to recognize an immunodominant region of gp120 centered on residue 241. Residue 241 occupies a hole in the glycan defenses of the BG505 isolate, with fewer than 3% of global isolates lacking a glycan site at this position. However, at least one conserved glycan site is missing in 89% of viruses, suggesting the presence of glycan holes in most HIV isolates. Serum evidence is consistent with targeting of holes in natural infection. The immunogenic nature of breaches in the glycan shield has been under-appreciated in previous attempts to understand autologous neutralizing antibody responses and has important potential consequences for HIV vaccine design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Jardine2016-hh" class="col-sm-8"> <div class="title">Minimally mutated HIV-1 broadly neutralizing antibodies to guide reductionist vaccine design</div> <div class="author"> Joseph G Jardine, Devin Sok, Jean-Philippe Julien, and <span class="more-authors" title="click to view 21 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '21 more authors' ? 'Bryan Briney, Anita Sarkar, Chi-Hui Liang, Erin A Scherer, Carole J Henry Dunand, Yumiko Adachi, Devan Diwanji, Jessica Hsueh, Meaghan Jones, Oleksandr Kalyuzhniy, Michael Kubitz, Skye Spencer, Matthias Pauthner, Karen L Saye-Francisco, Fabian Sesterhenn, Patrick C Wilson, Denise M Galloway, Robyn L Stanfield, Ian A Wilson, Dennis R Burton, William R Schief' : '21 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">21 more authors</span> </div> <div class="periodical"> <em>PLoS pathogens</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.ppat.1005815" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>An optimal HIV vaccine should induce broadly neutralizing antibodies (bnAbs) that neutralize diverse viral strains and subtypes. However, potent bnAbs develop in only a small fraction of HIV-infected individuals, all contain rare features such as extensive mutation, insertions, deletions, and/or long complementarity-determining regions, and some are polyreactive, casting doubt on whether bnAbs to HIV can be reliably induced by vaccination. We engineered two potent VRC01-class bnAbs that minimized rare features. According to a quantitative features frequency analysis, the set of features for one of these minimally mutated bnAbs compared favorably with all 68 HIV bnAbs analyzed and was similar to antibodies elicited by common vaccines. This same minimally mutated bnAb lacked polyreactivity in four different assays. We then divided the minimal mutations into spatial clusters and dissected the epitope components interacting with those clusters, by mutational and crystallographic analyses coupled with neutralization assays. Finally, by synthesizing available data, we developed a working-concept boosting strategy to select the mutation clusters in a logical order following a germline-targeting prime. We have thus developed potent HIV bnAbs that may be more tractable vaccine goals compared to existing bnAbs, and we have proposed a strategy to elicit them. This reductionist approach to vaccine design, guided by antibody and antigen structure, could be applied to design candidate vaccines for other HIV bnAbs or protective Abs against other pathogens.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sok2016-fq" class="col-sm-8"> <div class="title">A prominent site of antibody vulnerability on HIV envelope incorporates a motif associated with CCR5 binding and its camouflaging glycans</div> <div class="author"> Devin Sok, Matthias Pauthner, <em>Bryan Briney</em>, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Jeong Hyun Lee, Karen L Saye-Francisco, Jessica Hsueh, Alejandra Ramos, Khoa M Le, Meaghan Jones, Joseph G Jardine, Raiza Bastidas, Anita Sarkar, Chi-Hui Liang, Sachin S Shivatare, Chung-Yi Wu, William R Schief, Chi-Huey Wong, Ian A Wilson, Andrew B Ward, Jiang Zhu, Pascal Poignard, Dennis R Burton' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2016.06.026" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The dense patch of high-mannose-type glycans surrounding the N332 glycan on the HIV envelope glycoprotein (Env) is targeted by multiple broadly neutralizing antibodies (bnAbs). This region is relatively conserved, implying functional importance, the origins of which are not well understood. Here we describe the isolation of new bnAbs targeting this region. Examination of these and previously described antibodies to Env revealed that four different bnAb families targeted the (324)GDIR(327) peptide stretch at the base of the gp120 V3 loop and its nearby glycans. We found that this peptide stretch constitutes part of the CCR5 co-receptor binding site, with the high-mannose patch glycans serving to camouflage it from most antibodies. GDIR-glycan bnAbs, in contrast, bound both (324)GDIR(327) peptide residues and high-mannose patch glycans, which enabled broad reactivity against diverse HIV isolates. Thus, as for the CD4 binding site, bnAb effectiveness relies on circumventing the defenses of a critical functional region on Env.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Hicar2016-no" class="col-sm-8"> <div class="title">Human antibodies that recognize novel immunodominant Quaternary epitopes on the HIV-1 Env protein</div> <div class="author"> Mark D Hicar, Xuemin Chen, Chidananda Sulli, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Trevor Barnes, Jason Goodman, Hakimuddin Sojar, Bryan Briney, Jordan Willis, Valentine U Chukwuma, Spyros A Kalams, Benjamin J Doranz, Paul Spearman, James E Crowe' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>PloS one</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pone.0158861" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Numerous broadly neutralizing antibodies (Abs) target epitopes that are formed or enhanced during mature HIV envelope formation (i.e. quaternary epitopes). Generally, it is thought that Env epitopes that induce broadly neutralizing Abs are difficult to access and poorly immunogenic because of the characteristic oligomerization, conformational flexibility, sequence diversity and extensive glycosylation of Env protein. To enhance for isolation of quaternary epitope-targeting Abs (QtAbs), we previously used HIV virus-like particles (VLPs) to bind B cells from long-term non-progressor subjects to identify a panel of monoclonal Abs. When expressed as recombinant full-length Abs, a subset of these novel Abs exhibited the binding profiles of QtAbs, as they either failed to bind to monomeric Env protein or showed much higher affinity for Env trimers and VLPs. These QtAbs represented a significant proportion of the B-cell response identified with VLPs. The Ab genes of these clones were highly mutated, but they did not neutralize common HIV strains. We sought to further define the epitopes targeted by these QtAbs. Competition-binding and mapping studies revealed these Abs targeted four separate epitopes; they also failed to compete for binding by Abs to known major neutralizing epitopes. Detailed epitope mapping studies revealed that two of the four epitopes were located in the gp41 subunit of Env. These QtAbs bound pre-fusion forms of antigen and showed differential binding kinetics depending on whether oligomers were produced as recombinant gp140 trimers or as full-length Env incorporated into VLPs. Antigenic regions within gp41 present unexpectedly diverse structural epitopes, including these QtAb epitopes, which may be targeted by the naturally occurring Ab response to HIV infection.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="MacLeod2016-bs" class="col-sm-8"> <div class="title">Early antibody lineage diversification and independent limb maturation lead to broad HIV-1 neutralization targeting the env high-mannose patch</div> <div class="author"> Daniel T MacLeod, Nancy M Choi, <em>Bryan Briney</em>, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Fernando Garces, Lorena S Ver, Elise Landais, Ben Murrell, Terri Wrin, William Kilembe, Chi-Hui Liang, Alejandra Ramos, Chaoran B Bian, Lalinda Wickramasinghe, Leopold Kong, Kemal Eren, Chung-Yi Wu, Chi-Huey Wong, IAVI Protocol C Investigators &lt;sup&gt;&amp;&lt;/sup&gt; The IAVI African HIV Research Network, Sergei L Kosakovsky Pond, Ian A Wilson, Dennis R Burton, Pascal Poignard' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2016.04.016" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The high-mannose patch on HIV Env is a preferred target for broadly neutralizing antibodies (bnAbs), but to date, no vaccination regimen has elicited bnAbs against this region. Here, we present the development of a bnAb lineage targeting the high-mannose patch in an HIV-1 subtype-C-infected donor from sub-Saharan Africa. The Abs first acquired autologous neutralization, then gradually matured to achieve breadth. One Ab neutralized &gt;47% of HIV-1 strains with only ∼11% somatic hypermutation and no insertions or deletions. By sequencing autologous env, we determined key residues that triggered the lineage and participated in Ab-Env coevolution. Next-generation sequencing of the Ab repertoire showed an early expansive diversification of the lineage followed by independent maturation of individual limbs, several of them developing notable breadth and potency. Overall, the findings are encouraging from a vaccine standpoint and suggest immunization strategies mimicking the evolution of the entire high-mannose patch and promoting maturation of multiple diverse Ab pathways.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2016-eh" class="col-sm-8"> <div class="title">Clonify: unseeded antibody lineage assignment from next-generation sequencing data</div> <div class="author"> <em>Bryan Briney</em>, Khoa Le, Jiang Zhu, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Dennis R Burton' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Scientific reports</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/srep23901" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Defining the dynamics and maturation processes of antibody clonal lineages is crucial to understanding the humoral response to infection and immunization. Although individual antibody lineages have been previously analyzed in isolation, these studies provide only a narrow view of the total antibody response. Comprehensive study of antibody lineages has been limited by the lack of an accurate clonal lineage assignment algorithm capable of operating on next-generation sequencing datasets. To address this shortcoming, we developed Clonify, which is able to perform unseeded lineage assignment on very large sets of antibody sequences. Application of Clonify to IgG+ memory repertoires from healthy individuals revealed a surprising lack of influence of large extended lineages on the overall repertoire composition, indicating that this composition is driven less by the order and frequency of pathogen encounters than previously thought. Clonify is freely available at www.github.com/briney/clonify-python.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Willis2016-uo" class="col-sm-8"> <div class="title">Long antibody HCDR3s from HIV-naïve donors presented on a PG9 neutralizing antibody background mediate HIV neutralization</div> <div class="author"> Jordan R Willis, Jessica A Finn, <em>Bryan Briney</em>, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Gopal Sapparapu, Vidisha Singh, Hannah King, Celia C LaBranche, David C Montefiori, Jens Meiler, James E Crowe' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences of the United States of America</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1073/pnas.1518405113" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Development of broadly neutralizing antibodies (bnAbs) against HIV-1 usually requires prolonged infection and induction of Abs with unusual features, such as long heavy-chain complementarity-determining region 3 (HCDR3) loops. Here we sought to determine whether the repertoires of HIV-1-naïve individuals contain Abs with long HCDR3 loops that could mediate HIV-1 neutralization. We interrogated at massive scale the structural properties of long Ab HCDR3 loops in HIV-1-naïve donors, searching for structured HCDR3s similar to those of the HIV-1 bnAb PG9. We determined the nucleotide sequences encoding 2.3 × 10(7)unique HCDR3 amino acid regions from 70 different HIV-1-naïve donors. Of the 26,917 HCDR3 loops with 30-amino acid length identified, we tested 30 for further study that were predicted to have PG9-like structure when chimerized onto PG9. Three of these 30 PG9 chimeras bound to the HIV-1 gp120 monomer, and two were neutralizing. In addition, we found 14 naturally occurring HCDR3 sequences that acquired the ability to bind to the HIV-1 gp120 monomer when adding 2- to 7-amino acid mutations via computational design. Of those 14 designed Abs, 8 neutralized HIV-1, with IC50values ranging from 0.7 to 98 µg/mL. These data suggest that the repertoire of HIV-1-naïve individuals contains rare B cells that encode HCDR3 loops that bind or neutralize HIV-1 when presented on a PG9 background with relatively few or no additional mutations. Long HCDR3 sequences are present in the HIV-naïve B-cell repertoire, suggesting that this class of bnAbs is a favorable target for rationally designed preventative vaccine efforts.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Jardine2016-tt" class="col-sm-8"> <div class="title">HIV-1 broadly neutralizing antibody precursor B cells revealed by germline-targeting immunogen</div> <div class="author"> Joseph G Jardine, Daniel W Kulp, Colin Havenar-Daughton, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Anita Sarkar, Bryan Briney, Devin Sok, Fabian Sesterhenn, June Ereño-Orbea, Oleksandr Kalyuzhniy, Isaiah Deresa, Xiaozhen Hu, Skye Spencer, Meaghan Jones, Erik Georgeson, Yumiko Adachi, Michael Kubitz, Allan C deCamp, Jean-Philippe Julien, Ian A Wilson, Dennis R Burton, Shane Crotty, William R Schief' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em>Science (New York, N.Y.)</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.aad9195" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Induction of broadly neutralizing antibodies (bnAbs) is a major HIV vaccine goal. Germline-targeting immunogens aim to initiate bnAb induction by activating bnAb germline precursor B cells. Critical unmet challenges are to determine whether bnAb precursor naïve B cells bind germline-targeting immunogens and occur at sufficient frequency in humans for reliable vaccine responses. Using deep mutational scanning and multitarget optimization, we developed a germline-targeting immunogen (eOD-GT8) for diverse VRC01-class bnAbs. We then used the immunogen to isolate VRC01-class precursor naïve B cells from HIV-uninfected donors. Frequencies of true VRC01-class precursors, their structures, and their eOD-GT8 affinities support this immunogen as a candidate human vaccine prime. These methods could be applied to germline targeting for other classes of HIV bnAbs and for Abs to other pathogens.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Stapleton2016-fo" class="col-sm-8"> <div class="title">Haplotype-phased synthetic long reads from short-read sequencing</div> <div class="author"> James A Stapleton, Jeongwoon Kim, John P Hamilton, and <span class="more-authors" title="click to view 10 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '10 more authors' ? 'Ming Wu, Luiz C Irber, Rohan Maddamsetti, Bryan Briney, Linsey Newton, Dennis R Burton, C Titus Brown, Christina Chan, C Robin Buell, Timothy A Whitehead' : '10 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">10 more authors</span> </div> <div class="periodical"> <em>PloS one</em>, 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pone.0147229" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Next-generation DNA sequencing has revolutionized the study of biology. However, the short read lengths of the dominant instruments complicate assembly of complex genomes and haplotype phasing of mixtures of similar sequences. Here we demonstrate a method to reconstruct the sequences of individual nucleic acid molecules up to 11.6 kilobases in length from short (150-bp) reads. We show that our method can construct 99.97%-accurate synthetic reads from bacterial, plant, and animal genomic samples, full-length mRNA sequences from human cancer cell lines, and individual HIV env gene variants from a mixture. The preparation of multiple samples can be multiplexed into a single tube, further reducing effort and cost relative to competing approaches. Our approach generates sequencing libraries in three days from less than one microgram of DNA in a single-tube format without custom equipment or specialized expertise.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Andrabi2015-om" class="col-sm-8"> <div class="title">Identification of common features in prototype broadly neutralizing antibodies to HIV envelope V2 apex to facilitate vaccine design</div> <div class="author"> Raiees Andrabi, James E Voss, Chi-Hui Liang, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Bryan Briney, Laura E McCoy, Chung-Yi Wu, Chi-Huey Wong, Pascal Poignard, Dennis R Burton' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Immunity</em>, 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.immuni.2015.10.014" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Broadly neutralizing antibodies (bnAbs) directed to the V2 apex of the HIV envelope (Env) trimer isolated from individual HIV-infected donors potently neutralize diverse HIV strains, but strategies for designing immunogens to elicit bnAbs have not been identified. Here, we compared four prototypes (PG9, CH01, PGT145, and CAP256.VRC26.09) of V2 apex bnAbs and showed that all recognized a core epitope of basic V2 residues and the glycan-N160. Two prototype bnAbs were derived from VH-germlines that were 99% identical and used a common germline D-gene encoded YYD-motif to interact with the V2-epitope. We identified isolates that were neutralized by inferred germline (iGL) versions of three of the prototype bnAbs. Soluble Env derived from one of these isolates was shown to form a well-ordered Env trimer that could serve as an immunogen to initiate a V2-apex bnAb response. These studies illustrate a strategy to transition from panels of bnAbs to vaccine candidates.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Jardine2015-sm" class="col-sm-8"> <div class="title">HIV-1 VACCINES. Priming a broadly neutralizing antibody response to HIV-1 using a germline-targeting immunogen</div> <div class="author"> Joseph G Jardine, Takayuki Ota, Devin Sok, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Matthias Pauthner, Daniel W Kulp, Oleksandr Kalyuzhniy, Patrick D Skog, Theresa C Thinnes, Deepika Bhullar, Bryan Briney, Sergey Menis, Meaghan Jones, Mike Kubitz, Skye Spencer, Yumiko Adachi, Dennis R Burton, William R Schief, David Nemazee' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Science (New York, N.Y.)</em>, 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/science.aac5894" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A major goal of HIV-1 vaccine research is the design of immunogens capable of inducing broadly neutralizing antibodies (bnAbs) that bind to the viral envelope glycoprotein (Env). Poor binding of Env to unmutated precursors of bnAbs, including those of the VRC01 class, appears to be a major problem for bnAb induction. We engineered an immunogen that binds to VRC01-class bnAb precursors and immunized knock-in mice expressing germline-reverted VRC01 heavy chains. Induced antibodies showed characteristics of VRC01-class bnAbs, including a short CDRL3 (light-chain complementarity-determining region 3) and mutations that favored binding to near-native HIV-1 gp120 constructs. In contrast, native-like immunogens failed to activate VRC01-class precursors. The results suggest that rational epitope design can prime rare B cell precursors for affinity maturation to desired targets.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sok2014-ah" class="col-sm-8"> <div class="title">Recombinant HIV envelope trimer selects for quaternary-dependent antibodies targeting the trimer apex</div> <div class="author"> Devin Sok, Marit J Gils, Matthias Pauthner, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Jean-Philippe Julien, Karen L Saye-Francisco, Jessica Hsueh, Bryan Briney, Jeong Hyun Lee, Khoa M Le, Peter S Lee, Yuanzi Hua, Michael S Seaman, John P Moore, Andrew B Ward, Ian A Wilson, Rogier W Sanders, Dennis R Burton' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Proceedings of the National Academy of Sciences of the United States of America</em>, 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1073/pnas.1415789111" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Broadly neutralizing antibodies (bnAbs) targeting the trimer apex of HIV envelope are favored candidates for vaccine design and immunotherapy because of their great neutralization breadth and potency. However, methods of isolating bnAbs against this site have been limited by the quaternary nature of the epitope region. Here we report the use of a recombinant HIV envelope trimer, BG505 SOSIP.664 gp140, as an affinity reagent to isolate quaternary-dependent bnAbs from the peripheral blood mononuclear cells of a chronically infected donor. The newly isolated bnAbs, named “PGDM1400-1412,” show a wide range of neutralization breadth and potency. One of these variants, PGDM1400, is exceptionally broad and potent with cross-clade neutralization coverage of 83% at a median IC50 of 0.003 µg/mL. Overall, our results highlight the utility of BG505 SOSIP.664 gp140 as a tool for the isolation of quaternary-dependent antibodies and reveal a mosaic of antibody responses against the trimer apex within a clonal family.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2014-bt" class="col-sm-8"> <div class="title">Tissue-specific expressed antibody variable gene repertoires</div> <div class="author"> Bryan S Briney, Jordan R Willis, Jessica A Finn, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Brett A McKinney, James E Crowe' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>PloS one</em>, 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pone.0100839" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Recent developments in genetic technologies allow deep analysis of the sequence diversity of immune repertoires, but little work has been reported on the architecture of immune repertoires in mucosal tissues. Antibodies are the key to prevention of infections at the mucosal surface, but it is currently unclear whether the B cell repertoire at mucosal surfaces reflects the dominant antibodies found in the systemic compartment or whether mucosal tissues harbor unique repertoires. We examined the expressed antibody variable gene repertoires from 10 different human tissues using RNA samples derived from a large number of individuals. The results revealed that mucosal tissues such as stomach, intestine and lung possess unique antibody gene repertoires that differed substantially from those found in lymphoid tissues or peripheral blood. Mutation frequency analysis of mucosal tissue repertoires revealed that they were highly mutated, with little evidence for the presence of naïve B cells, in contrast to blood. Mucosal tissue repertoires possessed longer heavy chain complementarity determining region 3 loops than lymphoid tissue repertoires. We also noted a large increase in frequency of both insertions and deletions in the small intestine antibody repertoire. These data suggest that mucosal immune repertoires are distinct in many ways from the systemic compartment.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sok2014-pa" class="col-sm-8"> <div class="title">Promiscuous glycan site recognition by antibodies to the high-mannose patch of gp120 broadens neutralization of HIV</div> <div class="author"> Devin Sok, Katie J Doores, <em>Bryan Briney</em>, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Khoa M Le, Karen L Saye-Francisco, Alejandra Ramos, Daniel W Kulp, Jean-Philippe Julien, Sergey Menis, Lalinda Wickramasinghe, Michael S Seaman, William R Schief, Ian A Wilson, Pascal Poignard, Dennis R Burton' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Science translational medicine</em>, 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/scitranslmed.3008104" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Broadly neutralizing monoclonal antibodies (bnmAbs) that target the high-mannose patch centered around the glycan at position 332 on HIV Env are promising vaccine leads and therapeutic candidates because they effectively protect against mucosal SHIV challenge and strongly suppress SHIV viremia in established infection in macaque models. However, these antibodies demonstrate varying degrees of dependency on the N332 glycan site, and the origins of their neutralization breadth are not always obvious. By measuring neutralization on an extended range of glycan site viral variants, we found that some bnmAbs can use alternate N-linked glycans in the absence of the N332 glycan site and therefore neutralize a substantial number of viruses lacking the site. Furthermore, many of the antibodies can neutralize viruses in which the N332 glycan site is shifted to the 334 position. Finally, we found that a combination of three antibody families that target the high-mannose patch can lead to 99% neutralization coverage of a large panel of viruses containing the N332/N334 glycan site and up to 66% coverage for viruses that lack the N332/N334 glycan site. The results indicate that a diverse response against the high-mannose patch may provide near-equivalent coverage as a combination of bnmAbs targeting multiple epitopes. Additionally, the ability of some bnmAbs to use other N-linked glycan sites can help counter neutralization escape mediated by shifting of glycosylation sites. Overall, this work highlights the importance of promiscuous glycan binding properties in bnmAbs to the high-mannose patch for optimal antiviral activity in either protective or therapeutic modalities.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sok2013-ti" class="col-sm-8"> <div class="title">The Effects of Somatic Hypermutation on Neutralization and Binding in the PGT121 Family of Broadly Neutralizing HIV Antibodies</div> <div class="author"> Devin Sok, Uri Laserson, Jonathan Laserson, and <span class="more-authors" title="click to view 25 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '25 more authors' ? 'Yi Liu, Francois Vigneault, Jean-Philippe Julien, Bryan Briney, Alejandra Ramos, Karen F Saye, Khoa Le, Alison Mahan, Shenshen Wang, Mehran Kardar, Gur Yaari, Laura M Walker, Birgitte B Simen, Elizabeth P St. John, Po-Ying Chan-Hui, Kristine Swiderek, Stephen H Kleinstein, Galit Alter, Michael S Seaman, Arup K Chakraborty, Daphne Koller, Ian A Wilson, George M Church, Dennis R Burton, Pascal Poignard' : '25 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">25 more authors</span> </div> <div class="periodical"> <em>PLoS Pathogens</em>, 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.ppat.100375410.1371/journal.ppat.1003754.g00110.1371/journal.ppat.1003754.g00210.1371/journal.ppat.1003754.g00310.1371/journal.ppat.1003754.g00410.1371/journal.ppat.1003754.g00510.1371/journal.ppat.1003754.g00610.1371/journal.ppat.1003754.g00710.1371/journal.ppat.1003754.g00810.1371/journal.ppat.1003754.g00910.1371/journal.ppat.1003754.g01010.1371/journal.ppat.1003754.g01110.1371/journal.ppat.1003754.s00110.1371/journal.ppat.1003754.s00210.1371/journal.ppat.1003754.s00310.1371/journal.ppat.1003754.s00410.1371/journal.ppat.1003754.s00510.1371/journal.ppat.1003754.s00610.1371/journal.ppat.1003754.s00710.1371/journal.ppat.1003754.s00810.1371/journal.ppat.1003754.s00910.1371/journal.ppat.1003754.s01010.1371/journal.ppat.1003754.s01110.1371/journal.ppat.1003754.s01210.1371/journal.ppat.1003754.s01310.1371/journal.ppat.1003754.s01410.1371/journal.ppat.1003754.s01510.1371/journal.ppat.1003754.s01610.1371/journal.ppat.1003754.s01710.1371/journal.ppat.1003754.s01810.1371/journal.ppat.1003754.s01910.1371/journal.ppat.1003754.s02010.1371/journal.ppat.1003754.s02110.1371/journal.ppat.1003754.s02210.1371/journal.ppat.1003754.s02310.1371/journal.ppat.1003754.s02410.1371/journal.ppat.1003754.s02510.1371/journal.ppat.1003754.s02610.1371/journal.ppat.1003754.s027" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Broadly neutralizing HIV antibodies (bnAbs) are typically highly somatically mutated, raising doubts as to whether they can be elicited by vaccination. We used 454 sequencing and designed a novel phylogenetic method to model lineage evolution of the bnAbs PGT121-134 and found a positive correlation between the level of somatic hypermutation (SHM) and the development of neutralization breadth and potency. Strikingly, putative intermediates were characterized that show approximately half the mutation level of PGT121-134 but were still capable of neutralizing roughly 40-80% of PGT121-134 sensitive viruses in a 74-virus panel at median titers between 15- and 3-fold higher than PGT121-134. Such antibodies with lower levels of SHM may be more amenable to elicitation through vaccination while still providing noteworthy coverage. Binding characterization indicated a preference of inferred intermediates for native Env binding over monomeric gp120, suggesting that the PGT121-134 lineage may have been selected for binding to native Env at some point during maturation. Analysis of glycan-dependent neutralization for inferred intermediates identified additional adjacent glycans that comprise the epitope and suggests changes in glycan dependency or recognition over the course of affinity maturation for this lineage. Finally, patterns of neutralization of inferred bnAb intermediates suggest hypotheses as to how SHM may lead to potent and broad HIV neutralization and provide important clues for immunogen design.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Willis2013-fu" class="col-sm-8"> <div class="title">Human Germline Antibody Gene Segments Encode Polyspecific Antibodies</div> <div class="author"> Jordan R Willis, Bryan S Briney, Samuel L DeLuca, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'James E Crowe, Jens Meiler' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>PLoS Computational Biology</em>, 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pcbi.100304510.1371/journal.pcbi.1003045.g00110.1371/journal.pcbi.1003045.g00210.1371/journal.pcbi.1003045.g00310.1371/journal.pcbi.1003045.g00410.1371/journal.pcbi.1003045.g00510.1371/journal.pcbi.1003045.g00610.1371/journal.pcbi.1003045.t00110.1371/journal.pcbi.1003045.s00110.1371/journal.pcbi.1003045.s00210.1371/journal.pcbi.1003045.s00310.1371/journal.pcbi.1003045.s00410.1371/journal.pcbi.1003045.s005" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Structural flexibility in germline gene-encoded antibodies allows promiscuous binding to diverse antigens. The binding affinity and specificity for a particular epitope typically increase as antibody genes acquire somatic mutations in antigen-stimulated B cells. In this work, we investigated whether germline gene-encoded antibodies are optimal for polyspecificity by determining the basis for recognition of diverse antigens by antibodies encoded by three VH gene segments. Panels of somatically mutated antibodies encoded by a common VH gene, but each binding to a different antigen, were computationally redesigned to predict antibodies that could engage multiple antigens at once. The Rosetta multi-state design process predicted antibody sequences for the entire heavy chain variable region, including framework, CDR1, and CDR2 mutations. The predicted sequences matched the germline gene sequences to a remarkable degree, revealing by computational design the residues that are predicted to enable polyspecificity, i.e., binding of many unrelated antigens with a common sequence. The process thereby reverses antibody maturation in silico. In contrast, when designing antibodies to bind a single antigen, a sequence similar to that of the mature antibody sequence was returned, mimicking natural antibody maturation in silico. We demonstrated that the Rosetta computational design algorithm captures important aspects of antibody/antigen recognition. While the hypervariable region CDR3 often mediates much of the specificity of mature antibodies, we identified key positions in the VH gene encoding CDR1, CDR2, and the immunoglobulin framework that are critical contributors for polyspecificity in germline antibodies. Computational design of antibodies capable of binding multiple antigens may allow the rational design of antibodies that retain polyspecificity for diverse epitope binding.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2013-xa" class="col-sm-8"> <div class="title">Secondary mechanisms of diversification in the human antibody repertoire</div> <div class="author"> Bryan S Briney and James E Crowe Jr. </div> <div class="periodical"> <em>Frontiers in Immunology</em>, 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.3389/fimmu.2013.00042" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>V(D)J recombination and somatic hypermutation (SHM) are the primary mechanisms for diversification of the human antibody repertoire. These mechanisms allow for rapid humoral immune responses to a wide range of pathogenic challenges. V(D)J recombination efficiently generate a virtually limitless diversity through random recombination of variable (V), diversity (D), and joining (J) genes with diverse non-templated junctions between the selected gene segments. Following antigen stimulation, affinity maturation by SHM produces antibodies with refined specificity mediated by mutations typically focused in complementarity determining regions (CDRs), which form the bulk of the antigen recognition site. While V(D)J recombination and SHM are responsible for much of the diversity of the antibody repertoire, there are several secondary mechanisms that, while less frequent, make substantial contributions to antibody diversity including V(DD)J recombination (or D-D fusion), SHM-associated insertions and deletions, and affinity maturation and antigen contact by non-CDR regions of the antibody. In addition to enhanced diversity, these mechanisms allow the production of antibodies that are critical to response to a variety of viral and bacterial pathogens but that would be difficult to generate using only the primary mechanisms of diversification.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2012-kx" class="col-sm-8"> <div class="title">Location and length distribution of somatic hypermutation-associated DNA insertions and deletions reveals regions of antibody structural plasticity</div> <div class="author"> B S Briney, J R Willis, and J E Crowe </div> <div class="periodical"> <em>Genes &amp; Immunity</em>, 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/gene.2012.28" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Following the initial diversity generated by V(D)J recombination, somatic hypermutation is the principal mechanism for producing further antibody repertoire diversity in antigen-experienced B cells. While somatic hypermutation typically results in single-nucleotide substitutions, the infrequent incorporation of genetic insertions and deletions has also been associated with the somatic hypermutation process. We used high-throughput antibody sequencing to determine the sequence of thousands of antibody genes containing somatic hypermutation-associated insertions and deletions (SHA indels), which revealed significant differences between the location of SHA indels and somatic mutations. Further, we identified a cluster of insertions and deletions in the antibody framework 3 region, which corresponds to the hypervariable region 4 (HV4) in T-cell receptors. We propose that this HV4-like region, identified by SHA indel analysis, represents a region of under-appreciated affinity maturation potential. Finally, through the analysis of both location and length distribution of SHA indels, we have determined regions of structural plasticity within the antibody protein.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2012-xh" class="col-sm-8"> <div class="title">High-throughput antibody sequencing reveals genetic evidence of global regulation of the naïve and memory repertoires that extends across individuals</div> <div class="author"> B S Briney, J R Willis, B A McKinney, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'J E Crowe' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Genes and immunity</em>, 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/gene.2012.20" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Vast diversity in the antibody repertoire is a key component of the adaptive immune response. This diversity is generated centrally through the assembly of variable, diversity and joining gene segments, and peripherally by somatic hypermutation and class-switch recombination. The peripheral diversification process is thought to only occur in response to antigenic stimulus, producing antigen-selected memory B cells. Surprisingly, analyses of the variable, diversity and joining gene segments have revealed that the naïve and memory subsets are composed of similar proportions of these elements. Lacking, however, is a more detailed study, analyzing the repertoires of naïve and memory subsets at the level of the complete V(D)J recombinant. This report presents a thorough examination of V(D)J recombinants in the human peripheral blood repertoire, revealing surprisingly large repertoire differences between circulating B-cell subsets and providing genetic evidence for global control of repertoire diversity in naïve and memory circulating B-cell subsets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2012-eg" class="col-sm-8"> <div class="title">Frequency and genetic characterization of V(DD)J recombinants in the human peripheral blood antibody repertoire</div> <div class="author"> Bryan S Briney, Jordan R Willis, Mark D Hicar, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'James W Thomas, James E Crowe' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Immunology</em>, 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1111/j.1365-2567.2012.03605.x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Antibody heavy-chain recombination that results in the incorporation of multiple diversity (D) genes, although uncommon, contributes substantially to the diversity of the human antibody repertoire. Such recombination allows the generation of heavy chain complementarity determining region 3 (HCDR3) regions of extreme length and enables junctional regions that, because of the nucleotide bias of N-addition regions, are difficult to produce through normal V(D)J recombination. Although this non-classical recombination process has been observed infrequently, comprehensive analysis of the frequency and genetic characteristics of such events in the human peripheral blood antibody repertoire has not been possible because of the rarity of such recombinants and the limitations of traditional sequencing technologies. Here, through the use of high-throughput sequencing of the normal human peripheral blood antibody repertoire, we analysed the frequency and genetic characteristics of V(DD)J recombinants. We found that these recombinations were present in approximately 1 in 800 circulating B cells, and that the frequency was severely reduced in memory cell subsets. We also found that V(DD)J recombination can occur across the spectrum of diversity genes, indicating that virtually all recombination signal sequences that flank diversity genes are amenable to V(DD)J recombination. Finally, we observed a repertoire bias in the diversity gene repertoire at the upstream (5’) position, and discovered that this bias was primarily attributable to the order of diversity genes in the genomic locus.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2012-dk" class="col-sm-8"> <div class="title">High-throughput antibody sequencing reveals genetic evidence of global regulation of the naïve and memory repertoires that extends across individuals</div> <div class="author"> Bryan S Briney, Jordan R Willis, Brett A McKinney, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'James E Crowe' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Genes and immunity</em>, 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/gene.2012.20" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Vast diversity in the antibody repertoire is a key component of the adaptive immune response. This diversity is generated centrally through the assembly of variable, diversity and joining gene segments, and peripherally by somatic hypermutation and class-switch …</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Briney2012-vz" class="col-sm-8"> <div class="title">Human peripheral blood antibodies with long HCDR3s are established primarily at original recombination using a limited subset of germline genes</div> <div class="author"> Bryan S Briney, Jordan R Willis, and James E Crowe </div> <div class="periodical"> <em>PloS one</em>, 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1371/journal.pone.0036750" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>A number of antibodies that efficiently neutralize microbial targets contain long heavy chain complementarity determining region 3 (HCDR3) loops. For HIV, several of the most broad and potently neutralizing antibodies have exceptionally long HCDR3s. Two broad potently neutralizing HIV-specific antibodies, PG9 and PG16, exhibit secondary structure. Two other long HCDR3 antibodies, 2F5 and 4E10, protect against mucosal challenge with SHIV. Induction of such long HCDR3 antibodies may be critical to the design of an effective vaccine strategy for HIV and other pathogens, however it is unclear at present how to induce such antibodies. Here, we present genetic evidence that human peripheral blood antibodies containing long HCDR3s are not primarily generated by insertions introduced during the somatic hypermutation process. Instead, they are typically formed by processes occurring as part of the original recombination event. Thus, the response of B cells encoding antibodies with long HCDR3s results from selection of unusual clones from the naïve repertoire rather than through accumulation of insertions. These antibodies typically use a small subset of D and J gene segments that are particularly suited to encoding long HCDR3s, resulting in the incorporation of highly conserved genetic elements in the majority of antibody sequences encoding long HCDR3s.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Krause2011-ft" class="col-sm-8"> <div class="title">Epitope-Specific Human Influenza Antibody Repertoires Diversify by B Cell Intraclonal Sequence Divergence and Interclonal Convergence</div> <div class="author"> Jens C Krause, Tshidi Tsibane, Terrence M Tumpey, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Chelsey J Huffman, Bryan S Briney, Scott A Smith, Christopher F Basler, James E Crowe' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>The Journal of Immunology</em>, 2011 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.4049/jimmunol.1101823" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>We generated from a single blood sample five independent human mAbs that recognized the Sa antigenic site on the head of influenza hemagglutinin and exhibited inhibitory activity against a broad panel of H1N1 strains. All five Abs used the V(H)3-7 and J(H)6 gene segments, but at least four independent clones were identified by junctional analysis. High-throughput sequence analysis of circulating B cells revealed that each of the independent clones were members of complex phylogenetic lineages that had diversified widely using a pattern of progressive diversification through somatic mutation. Unexpectedly, B cells encoding multiple diverging lineages of these clones, including many containing very few mutations in the Ab genes, persisted in the circulation. Conversely, we noted frequent instances of amino acid sequence convergence in the Ag combining sites exhibited by members of independent clones, suggesting a strong selection for optimal binding sites. We suggest that maintenance in circulation of a wide diversity of somatic variants of dominant clones may facilitate recognition of drift variant virus epitopes that occur in rapidly mutating virus Ags, such as influenza hemagglutinin. In fact, these Ab clones recognize an epitope that acquired three glycosylation sites mediating escape from previously isolated human Abs.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Hicar2010-by" class="col-sm-8"> <div class="title">Pseudovirion Particles Bearing Native HIV Envelope Trimers Facilitate a Novel Method for Generating Human Neutralizing Monoclonal Antibodies Against HIV</div> <div class="author"> Mark D Hicar, Xuemin Chen, <em>Bryan Briney</em>, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Jason Hammonds, Jaang-Jiun Wang, Spyros Kalams, Paul W Spearman, James E Crowe' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>JAIDS Journal of Acquired Immune Deficiency Syndromes</em>, 2010 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1097/QAI.0b013e3181dc98a3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Monomeric HIV envelope vaccines fail to elicit broadly neutralizing antibodies or to protect against infection. Neutralizing antibodies against HIV bind to native functionally active Env trimers on the virion surface. Gag-Env pseudovirions recapitulate the native trimer and could serve as an effective epitope presentation platform for study of the neutralizing antibody response in HIV-infected individuals. To address if pseudovirions can recapitulate native HIV virion epitope structures, we carefully characterized these particles, concentrating on the antigenic structure of the coreceptor binding site. By blue native gel shift assays, Gag-Env pseudovirions were shown to contain native trimers that were competent for binding to neutralizing monoclonal antibodies. In enzyme-linked immunosorbent assay, pseudovirions exhibited increased binding of known CD4-induced antibodies after addition of CD4. Using flow cytometric analysis, fluorescently labeled pseudovirions specifically identified a subset of antigen-specific B cells in HIV-infected subjects. Interestingly, the sequence of one of these novel human antibodies, identified during cloning of single HIV-specific B cells and designated 2C6, exhibited homology to mAb 47e, a known anti-CD4-induced coreceptor binding site antibody. The secreted monoclonal antibody 2C6 did not bind monomeric gp120, but specifically bound envelope on pseudovirions. A recombinant form of the antibody 2C6 acted as a CD4-induced epitope-specific antibody in neutralization assays, yet did not bind monomeric gp120. These findings imply specificity against a quaternary epitope presented on the pseudovirion envelope spike. These data demonstrate that Gag-Env pseudovirions recapitulate CD4 and coreceptor binding pocket antigenic structures and can facilitate identification of B-cell clones that secrete neutralizing antibodies.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Briney Lab. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>